---
title: Macroâ€“ and microevolutionary insights into the evolution of foraging behavior and life history traits

author:

- \linebreak \linebreak Dylan Padilla \linebreak

date:

- Oral comprehensive exam \linebreak \linebreak \tiny Evolutionary Biology PhD Program \linebreak School of Life Sciences \linebreak July 18, 2023

output:
  beamer_presentation:
      incremental: true
      slide_level: 2
      includes:
         in_header: style.tex
aspectratio: 1610
---

```{r setup, include = FALSE}

library(rmarkdown)
library(xaringan)
library(xaringanExtra)
library(knitr)
library(icons)
library(emo)
library(lattice)
require(phylopath)
require(ggplot2)
require(gridExtra)
require(grid)
require(ggraph)
require(phylolm)

## preset code chunk options

knitr::opts_chunk$set(dpi = 300,            # ensures high resolution images
                      echo = FALSE,         # FALSE: code chunks don't show
                      warning = FALSE,      # FALSE: suppresses warning messages
                      error = FALSE,       # FALSE: Suppresses error messages
                      fig.align = "center")

```



***

## Outline

- Chapter 1: Macroclimatic and maternal effects on the evolution of reproductive traits in lizards

- Chapter 2: The correlated evolution of foraging mode and reproductive effort in lizards

- Chapter 3: Foraging efficiency of rover and sitter larvae of *Drosophila melanogaster*

- Chapter 4: Geographic variation of the *for* gene revealed signatures of local adaptation in *Drosophila melanogaster*


# Chapter 1:


***

## What is life history theory?

\begincols
  \begincol{.48\textwidth}

\justifying

![](imgs/hatchlings.jpeg){#id .class width=60% height=80%}
![](imgs/lizard_eggs.jpeg){#id .class width=70% height=70%}
<br><br>
  \endcol
\begincol{.48\textwidth}

\justifying

![](imgs/komodo_dragon.jpeg){#id .class width=80% height=70%}
![](imgs/phrynosoma.jpeg)

  \endcol
\endcols

***

## Optimal reproductive tactics

```{r, out.width="225px", fig.align = "center"}

knitr::include_graphics("imgs/hyp1.png")

```

***

## Optimal reproductive tactics

```{r, out.width="225px", fig.align = "center"}

knitr::include_graphics("imgs/hyp2.png")

```

***

## Optimal reproductive tactics

```{r, out.width="225px", fig.align = "center"}

knitr::include_graphics("imgs/hyp3.png")

```

***

## Relationships among environmental/body temperature and life-history traits

\begincols
  \begincol{.48\textwidth}


\justifying

![](imgs/rev_hyp.jpeg)

  \endcol
\begincol{.48\textwidth}

\justifying

![](imgs/tree4.jpeg)

  \endcol
\endcols


***

\centering

![](imgs/agamidae.jpeg){#id .class height=25% width=30%}
![](imgs/anguidae.jpeg){#id .class height=25% width=30%}
![](imgs/chamaeleonidae.jpeg){#id .class height=25% width=30%}
![](imgs/gekkonidae.jpeg){#id .class height=25% width=45%}
![](imgs/gymnophthalmidae.jpeg){#id .class height=25% width=30%}
![](imgs/dactyloidae.jpeg){#id .class height=25% width=30%}
![](imgs/iguanidae.jpeg){#id .class height=25% width=30%}
![](imgs/lacertidae.jpeg){#id .class height=25% width=30%}
![](imgs/sphaerodactylidae.jpeg){#id .class height=25% width=30%}
![](imgs/teiidae.jpeg){#id .class height=25% width=30%}
![](imgs/liolaemidae.jpeg){#id .class height=25% width=30%}
![](imgs/phrynosomatidae.jpeg){#id .class height=25% width=35%}


***

\centering

![](imgs/worldclim.png){#id .class height=25% width=70%} \linebreak
![](imgs/map2.png){#id .class height=50% width=50%} \linebreak \linebreak

Phylogenetic path analysis, package \textit{phylopath} in R

***

## A model describing a direct and an indirect effect of precipitation on the evolution of reproductive traits was strongly supported \linebreak

\begincols
  \begincol{.48\textwidth}


\justifying

![](imgs/mod_sel.jpeg)

  \endcol
\begincol{.48\textwidth}

\justifying

![](imgs/rev_pannel.jpeg)

  \endcol
\endcols


# Chapter 2:

***

## The foraging-mode paradigm \linebreak

\begincols
  \begincol{.48\textwidth}


\justifying

![](imgs/teiid.png){#id .class width=90% height=70%}
![](imgs/tiliqua.jpeg)
<br><br>
  \endcol
\begincol{.48\textwidth}

\justifying

![](imgs/sidewinder.png){#id .class width=90% height=80%}
![](imgs/snapping-turtle.jpeg)

  \endcol
\endcols


***

```{r, echo = FALSE, fig.height = 7, fig.width = 7, fig.align = "center", warning = FALSE, comment = " ", message = FALSE}


require(shape)

par(mar = c(1, 1, 1, 1))
plot(NA, NA, xlim = c(0, 10), ylim = c(-1, 10), ann = FALSE, axes = FALSE)


text(5, 10, "Foraging mode")
segments(3.5, 9.6, 6.5, 9.6, lwd = 2)
segments(3.5, 10.4, 6.5, 10.4, lwd = 2)
segments(3.5, 10.4, 3.5, 9.6, lwd = 2)
segments(6.5, 10.4, 6.5, 9.6, lwd = 2)
text(5, 7, "Distance traveled")
segments(3.5, 6.5, 6.5, 6.5, lwd = 2)
segments(3.5, 7.5, 6.5, 7.5, lwd = 2)
segments(3.5, 6.5, 3.5, 7.5, lwd = 2)
segments(6.5, 7.5, 6.5, 6.5, lwd = 2)
text(8.6, 7, "Predation risk")
segments(7.5, 6.5, 9.8, 6.5, lwd = 2)
segments(7.5, 7.5, 9.8, 7.5, lwd = 2)
segments(7.5, 6.5, 7.5, 7.5, lwd = 2)
segments(9.8, 7.5, 9.8, 6.5, lwd = 2)

Arrows(5, 9.5, 5, 7.8, lwd = 2, arr.type = "triangle")
Arrows(6.6, 7, 7, 7, lwd = 2, arr.type = "triangle")

```

***


```{r, echo = FALSE, fig.height = 7, fig.width = 7, fig.align = "center", warning = FALSE, comment = " ", message = FALSE}

par(mar = c(1, 1, 1, 1))
plot(NA, NA, xlim = c(0, 10), ylim = c(-1, 10), ann = FALSE, axes = FALSE)


text(5, 10, "Foraging mode")
segments(3.5, 9.6, 6.5, 9.6, lwd = 2)
segments(3.5, 10.4, 6.5, 10.4, lwd = 2)
segments(3.5, 10.4, 3.5, 9.6, lwd = 2)
segments(6.5, 10.4, 6.5, 9.6, lwd = 2)
text(5, 7, "Distance traveled")
segments(3.5, 6.5, 6.5, 6.5, lwd = 2)
segments(3.5, 7.5, 6.5, 7.5, lwd = 2)
segments(3.5, 6.5, 3.5, 7.5, lwd = 2)
segments(6.5, 7.5, 6.5, 6.5, lwd = 2)
text(8.6, 7, "Predation risk")
segments(7.5, 6.5, 9.8, 6.5, lwd = 2)
segments(7.5, 7.5, 9.8, 7.5, lwd = 2)
segments(7.5, 6.5, 7.5, 7.5, lwd = 2)
segments(9.8, 7.5, 9.8, 6.5, lwd = 2)
text(8.64, 3.7, "Reproductive effort")
segments(7.2, 4.2, 10.1, 4.2, lwd = 2)
segments(7.2, 3.2, 10.1, 3.2, lwd = 2)
segments(7.2, 4.2, 7.2, 3.2, lwd = 2)
segments(10.1, 3.2, 10.1, 4.2, lwd = 2)

Arrows(5, 9.5, 5, 7.8, lwd = 2, arr.type = "triangle")
Arrows(6.6, 7, 7, 7, lwd = 2, arr.type = "triangle")
Arrows(8.7, 4.8, 8.7, 6.1, lwd = 2, arr.type = "triangle")
Arrows(8.7, 6.1, 8.7, 4.8, lwd = 2, arr.type = "triangle")
Arrows(8.3, 4.76, 6.3, 6.2, lwd = 2, arr.type = "triangle")


```


***


```{r, echo = FALSE, fig.height = 7, fig.width = 7, fig.align = "center", warning = FALSE, comment = " ", message = FALSE}


par(mar = c(1, 1, 1, 1))
plot(NA, NA, xlim = c(0, 10), ylim = c(-1, 10), ann = FALSE, axes = FALSE)


text(5, 10, "Foraging mode")
segments(3.5, 9.6, 6.5, 9.6, lwd = 2)
segments(3.5, 10.4, 6.5, 10.4, lwd = 2)
segments(3.5, 10.4, 3.5, 9.6, lwd = 2)
segments(6.5, 10.4, 6.5, 9.6, lwd = 2)
text(5, 7, "Distance traveled")
segments(3.5, 6.5, 6.5, 6.5, lwd = 2)
segments(3.5, 7.5, 6.5, 7.5, lwd = 2)
segments(3.5, 6.5, 3.5, 7.5, lwd = 2)
segments(6.5, 7.5, 6.5, 6.5, lwd = 2)
text(8.6, 7, "Predation risk")
segments(7.5, 6.5, 9.8, 6.5, lwd = 2)
segments(7.5, 7.5, 9.8, 7.5, lwd = 2)
segments(7.5, 6.5, 7.5, 7.5, lwd = 2)
segments(9.8, 7.5, 9.8, 6.5, lwd = 2)
text(8.64, 3.7, "Reproductive effort")
segments(7.2, 4.2, 10.1, 4.2, lwd = 2)
segments(7.2, 3.2, 10.1, 3.2, lwd = 2)
segments(7.2, 4.2, 7.2, 3.2, lwd = 2)
segments(10.1, 3.2, 10.1, 4.2, lwd = 2)
text(4.97, 3.7, "Energy gain")
segments(3.7, 4.2, 6.2, 4.2, lwd = 2)
segments(3.7, 3.2, 6.2, 3.2, lwd = 2)
segments(3.7, 4.2, 3.7, 3.2, lwd = 2)
segments(6.2, 4.2, 6.2, 3.2, lwd = 2)


Arrows(5, 9.5, 5, 7.8, lwd = 2, arr.type = "triangle")
Arrows(6.6, 7, 7, 7, lwd = 2, arr.type = "triangle")
Arrows(8.7, 4.8, 8.7, 6.1, lwd = 2, arr.type = "triangle")
Arrows(8.7, 6.1, 8.7, 4.8, lwd = 2, arr.type = "triangle")
Arrows(8.3, 4.76, 6.3, 6.2, lwd = 2, arr.type = "triangle")
Arrows(5, 4.8, 5, 6.1, lwd = 2, arr.type = "triangle")
Arrows(5, 9.5, 5, 7.8, lwd = 2, arr.type = "triangle")
Arrows(5, 6.3, 5, 4.8, lwd = 2, arr.type = "triangle")

```

***


```{r, echo = FALSE, fig.height = 7, fig.width = 7, fig.align = "center", warning = FALSE, comment = " ", message = FALSE}

par(mar = c(1, 1, 1, 1))
plot(NA, NA, xlim = c(0, 10), ylim = c(-1, 10), ann = FALSE, axes = FALSE)
text(5, 10, "Foraging mode")
segments(3.5, 9.6, 6.5, 9.6, lwd = 2)
segments(3.5, 10.4, 6.5, 10.4, lwd = 2)
segments(3.5, 10.4, 3.5, 9.6, lwd = 2)
segments(6.5, 10.4, 6.5, 9.6, lwd = 2)
text(5, 7, "Distance traveled")
segments(3.5, 6.5, 6.5, 6.5, lwd = 2)
segments(3.5, 7.5, 6.5, 7.5, lwd = 2)
segments(3.5, 6.5, 3.5, 7.5, lwd = 2)
segments(6.5, 7.5, 6.5, 6.5, lwd = 2)
text(4.97, 3.7, "Energy gain")
segments(3.7, 4.2, 6.2, 4.2, lwd = 2)
segments(3.7, 3.2, 6.2, 3.2, lwd = 2)
segments(3.7, 4.2, 3.7, 3.2, lwd = 2)
segments(6.2, 4.2, 6.2, 3.2, lwd = 2)
text(8.6, 7, "Predation risk")
segments(7.5, 6.5, 9.8, 6.5, lwd = 2)
segments(7.5, 7.5, 9.8, 7.5, lwd = 2)
segments(7.5, 6.5, 7.5, 7.5, lwd = 2)
segments(9.8, 7.5, 9.8, 6.5, lwd = 2)
text(8.64, 3.7, "Reproductive effort")
segments(7.2, 4.2, 10.1, 4.2, lwd = 2)
segments(7.2, 3.2, 10.1, 3.2, lwd = 2)
segments(7.2, 4.2, 7.2, 3.2, lwd = 2)
segments(10.1, 3.2, 10.1, 4.2, lwd = 2)
text(1.57, 3.7, "Energy expenditure")
segments(0, 4.2, 3.1, 4.2, lwd = 2)
segments(0, 3.2, 3.1, 3.2, lwd = 2)
segments(0, 4.2, 0, 3.2, lwd = 2)
segments(3.1, 4.2, 3.1, 3.2, lwd = 2)
Arrows(5, 9.5, 5, 7.8, lwd = 2, arr.type = "triangle")
Arrows(5, 6.3, 5, 4.8, lwd = 2, arr.type = "triangle")
Arrows(5, 4.8, 5, 6.1, lwd = 2, arr.type = "triangle")
Arrows(3.8, 6.3, 2, 4.8, lwd = 2, arr.type = "triangle")
Arrows(8.3, 4.76, 6.3, 6.2, lwd = 2, arr.type = "triangle")
Arrows(8.7, 6.1, 8.7, 4.8, lwd = 2, arr.type = "triangle")
Arrows(8.7, 4.8, 8.7, 6.1, lwd = 2, arr.type = "triangle")
Arrows(6.6, 7, 7, 7, lwd = 2, arr.type = "triangle")
Arrows(7.1, 3.7, 6.5, 3.7, lwd = 2, arr.type = "triangle")
Arrows(6.5, 3.7, 7.0, 3.7, lwd = 2, arr.type = "triangle")


```


***

## Functional reproductive volume

\centering

![](imgs/model1.jpg){#id .class width=50% height=100%}


***

## Data source and description of variables

![](imgs/Meiri-2018.jpeg)


- Hatchling mass
- Offspring number
- Female mass
- Scaled-mass index


\[Reproductive\; output=Hatchling\; size \times Offspring\; number \]


***

## Ancestral reconstruction of foraging mode

\justifying

- We used a set of continuous-time, discrete-state Markov chain models to sample the character histories from their posterior probability distribution

- We fitted three different models to our data, using the function `make.simmap` from the `phytools` package of R v.1.0.1

- We estimated the prior distribution on the root node of the tree (`pi = 'estimated'`) and sampled the transition matrix, `Q`, from its posterior distribution (`Q = 'mcmc'`).


- We selected the most likely model of evolution based on the Akaike information criterion (AIC)

***

## Effects of maternal mass and foraging mode on reproductive output

\justifying

- We used PGLS to model the relationship among maternal mass, foraging mode and reproductive output through the `gls` function from the `nlme` package of R v.3.1.153

- The slope of the linear relationship can be interpreted as reproductive effortâ€”proportion of mass allocated to reproductionâ€”which enabled us to avoid statistical issues associated with the analysis of ratios


- Models in which maternal mass was standardized (M) enabled us to deal with outliers, thus increasing the robustness of our conclusions


***

\begincols
  \begincol{.48\textwidth}


\centering

![](imgs/foraging-tree.jpeg)

  \endcol
\begincol{.48\textwidth}

1. Sit-and-wait foraging is the most likely ancestral state \linebreak
2. Foraging mode is conserved among lizards

  \endcol
\endcols

***

## The evolution of reproductive effort in lizards was driven by an interaction between maternal mass and foraging mode

\centering

![](imgs/Figure3.jpeg){#id .class height=80% width=80%}

***

```{r, out.width = "65%"}


## A


bertalanffy <- function(la, k, t){
    la <- as.numeric(la)
    k <- as.numeric(k)
    t <- as.numeric(t)
    la*(1 - 4*exp(-k*t))
}
fem1 <- bertalanffy(la = 4, k = 0.3, t = seq(1, 35, 1))
fem2 <- bertalanffy(la = 4.9, k = 0.25, t = seq(1, 35, 1))
par(mar = c(7, 7, 2, 2))
plot(fem1 - 0.7 ~ seq(1, 35, 1), xlim = c(-2, 35), ylim = c(-6, 8), type = "l", lwd = 4, xaxt = "n", yaxt = "n", ylab = "Cumulative energy gain (m)", xlab = "time (t)", cex.lab = 2)
box(lwd = 2)
lines(seq(1, 35, 1) - 0.48, fem2, lwd = 4, lty = 4)
legend("topleft", legend = c("large female", "small female"), lty = c(4, 1), lwd = c(4, 4), bty = "n", cex = 1)


```


***


```{r, out.width = "65%"}

## B
fem3 <- bertalanffy(la = 5, k = 0.082, t = seq(1, 35, 1))
fem4 <- bertalanffy(la = 10, k = 0.21, t = seq(1, 35, 1))
par(mar = c(7, 7, 2, 2))
plot(fem3 - 0.7 ~ seq(1, 35, 1), xlim = c(4, 35), ylim = c(-6, 5), type = "l", lwd = 4,  col = "skyblue", xaxt = "n", yaxt = "n", ylab = "Cumulative energy gain (m)", xlab = " ", cex.lab = 2)
box(lwd = 2)
lines(seq(1, 35, 1), fem4 - 8, lwd = 4, lty = 2, col = "purple")
legend("topleft", legend = c("sit and wait", "widely foraging"), lty = c(1, 2), lwd = c(4, 4), col = c("skyblue", "purple"), bty = "n", cex = 1)
mtext("time (t)", side = 1, at = 25, line = 0.9, cex = 2)

```


***


```{r, out.width = "65%"}

## B
fem3 <- bertalanffy(la = 5, k = 0.082, t = seq(1, 35, 1))
fem4 <- bertalanffy(la = 10, k = 0.21, t = seq(1, 35, 1))
par(mar = c(7, 7, 2, 2))
plot(fem3 - 0.7 ~ seq(1, 35, 1), xlim = c(4, 35), ylim = c(-6, 5), type = "l", lwd = 4,  col = "skyblue", xaxt = "n", yaxt = "n", ylab = "Cumulative energy gain (m)", xlab = " ", cex.lab = 2)
box(lwd = 2)
lines(seq(1, 35, 1), fem4 - 8, lwd = 4, lty = 2, col = "purple")
segments(2.6, -6.5, 25, 6.2, lwd = 4)
segments(2.6, -6.5, 33.2, 6, lwd = 4)
legend("topleft", legend = c("sit and wait", "widely foraging"), lty = c(1, 2), lwd = c(4, 4), col = c("skyblue", "purple"), bty = "n", cex = 1)
mtext("time (t)", side = 1, at = 25, line = 0.9, cex = 2)

```



***


```{r, out.width = "65%", comment = FALSE, fig.align = "center"}

## B

fem3 <- bertalanffy(la = 5, k = 0.082, t = seq(1, 35, 1))
fem4 <- bertalanffy(la = 10, k = 0.21, t = seq(1, 35, 1))
par(mar = c(7, 7, 2, 2))
plot(fem3 - 0.7 ~ seq(1, 35, 1), xlim = c(4, 35), ylim = c(-6, 5), type = "l", lwd = 4,  col = "skyblue", xaxt = "n", yaxt = "n", ylab = "Cumulative energy gain (m)", xlab = " ", cex.lab = 2)
box(lwd = 2)
lines(seq(1, 35, 1), fem4 - 8, lwd = 4, lty = 2, col = "purple")
segments(2.6, -6.5, 25, 6.2, lwd = 4)
segments(2.6, -6.5, 33.2, 6, lwd = 4)
segments(13.2, -0.5, 2.6, -0.5, lwd = 4, lty = 2, col = "purple")
segments(15.9, -1.1, 2.6, -1.1, lwd = 4, col = "skyblue")
segments(13.2, -0.5, 13.2, -7, lwd = 4, lty = 2, col = "purple")
segments(15.9, -1.1, 15.9, -7, lwd = 4, col = "skyblue")
legend("topleft", legend = c("sit and wait", "widely foraging"), lty = c(1, 2), lwd = c(4, 4), col = c("skyblue", "purple"), bty = "n", cex = 1)
mtext(expression("t"[wf]), side = 1, at = 13.2, line = 0.5, cex = 1.8, font = 2)
mtext(expression("t"[sw]), side = 1, at = 17.2, line = 0.5, cex = 1.8, font = 2)
mtext("time (t)", side = 1, at = 25, line = 0.9, cex = 2)

```


***



```{r, out.width = "65%"}

## C

par(mar = c(7, 7, 2, 2))
curve(2 - log(x)^2, ylim = c(-0.5, 3), xlim = c(0.1, 1), lwd = 4, lty = 2, col = "purple", xaxt = "n", yaxt = "n", ylab = "Cumulative energy gain (m)", xlab = "time (t)", cex.lab = 2)
box(lwd = 2)
curve(log(x) + 1, add = TRUE, lwd = 4, col = "skyblue")
legend("topleft", legend = c("sit and wait", "widely foraging"), lty = c(1, 2), lwd = c(4, 4), col = c("skyblue", "purple"), bty = "n", cex = 1)


```

# Chapter 3:

***

## \textit{Drosophila melanogaster} foraging behavior

\centering

![](imgs/for-gene.png)
![](imgs/larva.png){#id .class height=30% width=50%} ![](imgs/paths.png){#id .class height=50% width=70%}


***

## Experimental design


\begincols
  \begincol{.48\textwidth}


\justifying


![](imgs/life-cycle.png)

  \endcol
\begincol{.48\textwidth}

\justifying

![](imgs/experimental-design.png)


  \endcol
\endcols


***

\begincols
  \begincol{.48\textwidth}


\justifying


```{r, out.width = "80%", comment = FALSE, fig.align = "center", fig.cap = "Schematic represention of an arena with a multi-patch distribution of resources.", dpi = 300}


pat <- runif(200, min = 0, max = 300)

layout(matrix(c(1, 2, 3,
                4, 5, 6,
                7, 8, 9), nrow = 3, ncol = 3, byrow = TRUE))

par(bg = "black", mar = c(3, 1, 1, 2))

# Change the plot region color
#rect(par("usr")[1], par("usr")[3],
#     par("usr")[2], par("usr")[4],
#     col = "black") # Color

for(i in 1:9){

    plot(pat, type = "p", pch = 16, col = "orange", axes = FALSE, ann = FALSE)
}

```


  \endcol
\begincol{.48\textwidth}

\justifying


```{r, out.width = "80%", comment = FALSE, fig.align = "center", fig.cap = "Schematic representation of an arena with a hypothetical uniform distribution of resources.", dpi = 300}


unif <- runif(1000, min = 0, max = 100)

# Create basic plot
plot(unif, type = "n", axes = FALSE, ann = FALSE)

# Change the plot region color
rect(par("usr")[1], par("usr")[3],
     par("usr")[2], par("usr")[4],
     col = "black") # Color

# Add a new plot
par(new = TRUE)

plot(unif, type = "p", pch = 16, col = "orange", axes = FALSE, ann = FALSE)

```


  \endcol
\endcols



***

## Predictions

```{r, out.width = "250px"}

energy <- c(4, 5, 6, 4, 5, 6, 3, 3, 5, 4, 4, 4)
pheno <- c(rep("rover", 3), rep("sitter", 3), rep("rover", 3), rep("sitter", 3))
food <- c(rep("uniform", 6), rep("patchy", 6))

mat <- cbind(pheno, food, energy)
mat <- as.data.frame(mat)
mat$energy <- as.numeric(mat$energy)

barchart(energy ~ pheno | food, data = mat, col = c("skyblue", "purple"), ylab = "Food consumption / time", scales= list(y = list(draw = FALSE)))

```

Assuming there is a GxE interaction, two possible outcomes are expected: In the left pannel, a rover genotype would comsume more food than a sitter genotype. In the right pannel, however, both rovers and sitters would consume the same amount of food

# Chapter 4:


***

## The Drosophila \textit{foraging} gene provides an opportunity to understand the mechanisms underlying evolutionary responses to environmental variation

\begincols
  \begincol{.48\textwidth}


\justifying

1. Given its allelic variants, one should expect geographic variation of the \textit{for} gene among populations

  \endcol
\begincol{.48\textwidth}

\justifying

![](imgs/cline.png)

  \endcol
\endcols


***

## The Drosophila \textit{foraging} gene provides an opportunity to understand the mechanisms underlying evolutionary responses to environmental variation

\begincols
  \begincol{.48\textwidth}


\justifying

2. The DEST dataset enables one to study genetic variation accross populations spanning America and Europe

  \endcol
\begincol{.48\textwidth}

\justifying

![](imgs/map-drosophila.png)

  \endcol
\endcols


***

## A genetic differentiation test, and an analysis of population structure revealed an east-west gradient in allele frequency \linebreak

\begincols
  \begincol{.48\textwidth}


\justifying

![](imgs/heatmap.png)

  \endcol
\begincol{.48\textwidth}

\justifying

![](imgs/admixture.png){#id .class heigth=90% width=90%}

  \endcol
\endcols


***

## Stronger structure in populations collected in America than those collected in Europe, although the structure of pools from Ukraine, Turkey, and Russia stands out

\centering

![](imgs/map.png){#id .class heigth=70% width=70%}


***

## Spatially varying selection driven by the seasonality of net primary production

\centering

![](imgs/cca.png){#id .class heigth=50% width=50%}

***

## Models of isolation by environment and isolation by distance are likely driving genetic differentiation among populations \linebreak

\begincols
  \begincol{.48\textwidth}


\justifying

![](imgs/adaptation.png){#id .class heigth=90% width=90%}

  \endcol
\begincol{.48\textwidth}

\justifying

![](imgs/distance.png){#id .class heigth=90% width=90%}

  \endcol
\endcols

***

![](imgs/eco-evo.png){#id .class height=60%}
![](imgs/proceedings.png){#id .class height=70%}

***

![](imgs/jeb.png)

***

\centering

Time for discussion!
