<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width">
<meta property="og:title" content="index.knit" />






<meta name="description" content="index.knit">

<script id="pandoc-meta" type="application/json">
{"newpage_html_class":"page-break-after","csl":"apa-numeric-superscript-brackets.csl","title":"<font size = '5'>\n<center>\nMacro‚Äî and microevolutionary insigths into the evolution of foraging behavior and the life history </font><br><br><br><br><font size = '4'> Research prospectus and written comprehensive exam </font><font size = '3'><br><br><br><br><br><br>Dylan Padilla<br>‚úâÔ∏è¬†<a href=\"mailto:dpadil10@asu.edu\" class=\"email\">dpadil10@asu.edu</a><br>üåç¬†<a href=\"https://dylanpadilla.netlify.app/\" class=\"uri\">https://dylanpadilla.netlify.app/</a><br><br><br><br><br><br>Evolutionary Biology PhD program<br>School of Life Sciences, Arizona State University, Tempe, AZ 85282, USA.<br><br><br><br><br>2024-04-13<br><br><br><img src = 'imgs/asu_logo.png' width = 165, height = 140>\n</center>\n</font>","lof":true,"toc-title":"Table of contents","output":{"pagedown::html_paged":{"self_contained":false,"css":["default-fonts","default","custom-pages.css"],"toc":true}},"bibliography":"packages.bib","link-citations":"no"}
</script>

<title>index.knit</title>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function() {
  // Retrieve previous config object if defined
  window.PagedConfig = window.PagedConfig || {};
  let beforePaged = PagedConfig.before;
  window.PagedConfig.before = async () => {
    if (beforePaged) await beforePaged();
    return new Promise((resolve, reject) => {
      var script = document.createElement("script");
      script.type = "text/javascript";
      var src = `https://mathjax.rstudio.com/latest/MathJax.js` + "?config=TeX-MML-AM_CHTML";
      if (location.protocol !== "file:" && /^https?:/.test(src))
        src = src.replace(/^https?:/, '');
      script.src = src;
      window.MathJax = {
        AuthorInit: () => {
          MathJax.Hub.Register.StartupHook("Begin", () => {
            MathJax.Hub.Queue(resolve);
          });
        }
      };
      document.getElementsByTagName("head")[0].appendChild(script);
    });
  };
})();
</script>

<link href="index_files/paged-0.20/css/default-fonts.css" rel="stylesheet" />
<link href="index_files/paged-0.20/css/default.css" rel="stylesheet" />
<script src="index_files/paged-0.20/js/config.js"></script>
<script src="index_files/paged-0.20/js/paged.js"></script>
<script src="index_files/paged-0.20/js/hooks.js"></script>
<script src="index_files/kePrint-0.0.1/kePrint.js"></script>
<link href="index_files/lightable-0.0.1/lightable.css" rel="stylesheet" />




<link rel="stylesheet" href="custom-pages.css" type="text/css" />

</head>

<body>

<div class="running-h1-title"></div>

<div class="running-h2-title"></div>



<div class="front-page">
<div id="header" class="title-page">
<h1 class="title"><font size = '5'>
<center>
Macro‚Äî and microevolutionary insigths into the evolution of foraging behavior and the life history </font><br><br><br><br><font size = '4'> Research prospectus and written comprehensive exam </font><font size = '3'><br><br><br><br><br><br>Dylan Padilla<br>‚úâÔ∏è¬†<a href="mailto:dpadil10@asu.edu" class="email">dpadil10@asu.edu</a><br>üåç¬†<a href="https://dylanpadilla.netlify.app/" class="uri">https://dylanpadilla.netlify.app/</a><br><br><br><br><br><br>Evolutionary Biology PhD program<br>School of Life Sciences, Arizona State University, Tempe, AZ 85282, USA.<br><br><br><br><br>2024-04-13<br><br><br><img src = 'imgs/asu_logo.png' width = 165, height = 140>
</center>
</font></h1>
</div>
</div>

<div class="front-matter-container">
<div id="TOC" class="level1 toc front-matter">
<h1 class="toc-title">Table of contents</h1>
<ul>
<li><a href="#LOF">List of Figures</a></li>
<li><a href="#background">Background</a>
<ul>
<li><a href="#current-state-of-knowledge">Current state of knowledge</a></li>
</ul></li>
<li><a href="#chapter-macroclimatic-and-maternal-effects-on-the-evolution-of-reproductive-traits-in-lizards"><span class="toc-section-number">1</span> Chapter: Macroclimatic and maternal effects on the evolution of reproductive traits in lizards</a>
<ul>
<li><a href="#introduction"><span class="toc-section-number">1.1</span> Introduction</a></li>
<li><a href="#materials-and-methods"><span class="toc-section-number">1.2</span> Materials and Methods</a></li>
<li><a href="#statistical-analyses"><span class="toc-section-number">1.3</span> Statistical analyses</a></li>
<li><a href="#results"><span class="toc-section-number">1.4</span> Results</a></li>
<li><a href="#discussion"><span class="toc-section-number">1.5</span> Discussion</a></li>
</ul></li>
<li><a href="#chapter-the-correlated-evolution-of-foraging-mode-and-reproductive-effort-in-lizards"><span class="toc-section-number">2</span> Chapter: The correlated evolution of foraging mode and reproductive effort in lizards</a>
<ul>
<li><a href="#introduction-1"><span class="toc-section-number">2.1</span> Introduction</a></li>
<li><a href="#materials-and-methods-1"><span class="toc-section-number">2.2</span> Materials and Methods</a></li>
<li><a href="#statistical-analyses-1"><span class="toc-section-number">2.3</span> Statistical analyses</a></li>
<li><a href="#results-1"><span class="toc-section-number">2.4</span> Results</a></li>
<li><a href="#discussion-1"><span class="toc-section-number">2.5</span> Discussion</a></li>
</ul></li>
<li><a href="#chapter-foraging-efficiency-of-rover-and-sitter-larvae-of-drosophila-melanogaster"><span class="toc-section-number">3</span> Chapter: Foraging efficiency of rover and sitter larvae of <em>Drosophila melanogaster</em></a>
<ul>
<li><a href="#introduction-2"><span class="toc-section-number">3.1</span> Introduction</a></li>
<li><a href="#experimental-design"><span class="toc-section-number">3.2</span> Experimental design</a></li>
<li><a href="#prediction"><span class="toc-section-number">3.3</span> Prediction</a></li>
</ul></li>
<li><a href="#chapter-geographic-variation-of-the-for-gene-revealed-signatures-of-local-adaptation-in-drosophila-melanogaster"><span class="toc-section-number">4</span> Chapter: Geographic variation of the <em>for</em> gene revealed signatures of local adaptation in <em>Drosophila melanogaster</em></a>
<ul>
<li><a href="#introduction-3"><span class="toc-section-number">4.1</span> Introduction</a></li>
<li><a href="#materials-and-methods-2"><span class="toc-section-number">4.2</span> Materials and Methods</a></li>
<li><a href="#results-2"><span class="toc-section-number">4.3</span> Results</a></li>
<li><a href="#discussion-2"><span class="toc-section-number">4.4</span> Discussion</a></li>
</ul></li>
<li><a href="#references">References</a></li>
</ul>
</div>
</div>

<div class="main">
<div id="LOF" class="section level1 lof unnumbered front-matter">
<h1>List of Figures</h1>
<ul>
<li><a href="#fig:fig1">1.1</a> Path models depicting relationships among body temperature and life-history traits derived from optimality models. Abbreviation in the path diagrams are as follows: BT = body temperature; CS = clutch size; FS = maternal length; HS = hatchling/neonate length; PP = primary production; PR = annual precipitation.</li>
<li><a href="#fig:fig2">1.2</a> Best-fit models of the evolution of reproductive traits considering body temperature (a)  and environmental temperature (b)  as independent variable. Thicker arrows indicate stronger effects. Values in parentheses denote the standard errors of the path coefficients.</li>
<li><a href="#fig:fig3">1.3</a> Evolution of the reproductive effort among 669 species of lizards. The dataset used to perform path analyses was reduced to 486 species because we could not extract data of climatic variables for all species present in the phylogeny.</li>
<li><a href="#fig:fig4">1.4</a> Relative importance of the causal models describing the evolution of reproductive traits in lizards.</li>
<li><a href="#fig:fig5">2.1</a> A conceptual model depicting putative relationships among foraging behavior, energetics, predation risk, and reproductive effort. The predicted relationships were derived from theoretical models of life-history evolution.</li>
<li><a href="#fig:fig6">2.2</a> Random sample of stochastic character maps depicting the evolution of foraging mode in 485 species of lizards. Bars at the tips of the phylogeny represent log-transformed values of reproductive effort for all lizards, but not the outgroup, Sphenodon punctatus. Pie charts on internal nodes represent posterior probability estimates resulting from 1,000 simulations of the character histories. Major clades are enumerated as follows: 1)  Gekkota, 2)  Scincoidea, 3)  Lacertoidea, 4)  Anguimorpha, 5)  Toxicofera, and 6)  Iguania. Lizard photos by Mark O‚ÄôShea.</li>
<li><a href="#fig:fig7">2.3</a> Effects of maternal mass and foraging mode on the evolution of reproductive effort of lizards, as determined by phylogenetic generalized least squares analysis. (a)  Difference in reproductive effort among foraging modes assuming body mass as a proxy for energy. (b)  Difference in reproductive effort among foraging modes assuming the scaled mass index as a proxy for energy.</li>
<li><a href="#fig:fig8">3.1</a> A theoretical model relating the cumulative energy gain m, as a function of time spent foraging, t. (a)  Larger larvae reach their maximum capacity at a higher value of m than small larvae. (b)  Rovers that are smaller but more efficient foragers may grow faster at a smaller size than larger and less efficient sitter larvae. t_{r} and t_{s} in (b)  represent the optimal foraging time of rover and sitter, respectively.</li>
<li><a href="#fig:fig9">3.2</a> Schematic representation of an arena with a hypothetical uniform distribution of resources.</li>
<li><a href="#fig:fig10">3.3</a> Schematic represention of an arena with a multi-patch distribution of resources.</li>
<li><a href="#fig:fig11">3.4</a> Assuming there is a GXE interaction, two possible outcomes are expected: In the left pannel, a rover genotype would comsume more food than a sitter genotype. In the right pannel, however, both rovers and sitters would consume the same amount of food.</li>
<li><a href="#fig:fig12">4.1</a> Latitudinal life-history clines in D. melanogaster. On multiple continents and subcontinents, spanning temperate to subtropical/tropical regions, fly populations exhibit major differences in fitness components across latitudes.</li>
<li><a href="#fig:fig13">4.2</a> Available data of genome-wide allele frequency estimates from &gt;270 population-based samples of D. melanogaster worldwide. DGN: samples available from the Drosophila Genome Nexus project; DrosEU: samples collected by the European Drosophila Population Genomics Consortium; DrosRTEC: samples collected by the Real Time Evolution Consortium.</li>
<li><a href="#fig:fig14">4.3</a> Heatmap depicting genetic differentiation among populations of D. melanogaster based on pairwise F_{st} values. In some cases, samples from adjacent localities were combined to avoid low sample sizes. Abbreviations are as follows: PA = Pennsylvania; MI-WI = Michigan and Wisconsin; NY-MA = New York and Massachusetts.</li>
<li><a href="#fig:fig15">4.4</a> Patterns of isolation by distance and isolation by environment according to Partial Mantel tests and Correlograms. (a-b)  Geographic and net primary production distances as a function of genetic distance. Colors represent estimated probability densities and the red line a smoothed local mean. (c-d)  Mantel correlation at different distance classes as a function of geographic and net primary production distances. Filled dots indicate significant correlations.</li>
<li><a href="#fig:fig16">4.5</a> Manhattan plot of the distribution of F_{st} loci as a function of their heterozygosity. No F_{st} loci showed significant deviation from the genetic background.</li>
<li><a href="#fig:fig17">4.6</a> Population structure analysis based on individual ancestry coefficients for a number of ancestral populations. (a)  Cross-entropy values for each snmf run with k ranging between k=1 and k=10. The red arrow indicates the most likely value of k. (b)  Admixture proportion across populations of D. melanogaster. Colors indicate genetic clusters.</li>
<li><a href="#fig:fig18">4.7</a> Mean admixture proportions across populations of D. melanogaster surveyed in America and Europe. Colors in the pie charts represent genetic clusters.</li>
<li><a href="#fig:fig19">4.8</a> Genotype-environment association test based on a latent factor mixed model with ridge penalty. (a)  Distribution of adjusted p-values using the default genomic inflation factor. (b)  Manhattan plot of loci (SNPs)  potentially affected by the PC1 predictor variable. Loci highlighted in red were considered to be under selection according to a False Discovery Rate of 0.05.</li>
</ul>
</div>
<div id="background" class="section level1 unnumbered">
<h1>Background</h1>
<body>
<p align="justify">
Life-history theory consists of models designed to explain how environmental factors shape the evolution of survival, growth, and reproduction.<span class="citation"><sup>[1,2]</sup></span> Much of this theory focuses on the covariation of body size and reproductive traits, given a tradeoff between the size and number of offspring.<span class="citation"><sup>[3]</sup></span> The optimal covariation depends on ways in which body size affects the availability of energy, the constraints on reproduction, and the survival of offspring.<span class="citation"><sup>[4‚Äì6]</sup></span> Often, larger females have more resources to spare, can carry more offspring during gestation, pass larger offspring during birth, or provide better care after birth. These factors influence the constraints on or the benefit of specific reproductive tactics. For example, when the abdominal cavity and pelvic opening constrain offspring size, larger females can produce larger offspring. Conversely, a small female may be unable to produce an offspring of the optimal size when this size exceeds the physical constraint.<span class="citation"><sup>[7‚Äì11]</sup></span> This constraint results in a positive relationship between maternal size and offspring size,<span class="citation"><sup>[12]</sup></span> in which each female produces the largest egg possible for its body size.
</p>
<p align="justify">
Natural selection can produce a similar relationship when larger females have greater surplus energy.<span class="citation"><sup>[6]</sup></span> Depending on both its foraging efficiency and the availability of resources, a larger female should use its additional energy to produce larger clutches. However, given a tradeoff between offspring size and offspring number, the investment in more offspring may be balanced by a decrease in the size of offspring. Similarly, when increasing the number of offspring increases competition among the resulting siblings, a female should use additional energy to produce larger offspring instead of more offspring. Interestingly, a mixed strategy can be optimal when intraspecific competition and offspring size both exert important influences on offspring survival; in such cases, larger females should lay larger offspring and larger clutches.<span class="citation"><sup>[6]</sup></span>
</p>
<p align="justify">
Because growth and body size are affected by environmental conditions, the covariation among abiotic factors should shape the evolution of life-history traits. Temperature is a well-known factor that directly and indirectly affects the evolution of the life history. The direct effect occurs when environmental conditions determine the relationship between the size and performance of offspring. Warmer environments often promote a high relative growth rate, potentially reducing the minimal size of offspring. In such cases, a female can produce more, smaller eggs as the optimal reproductive tactic. The indirect effect occurs because temperature greatly affects the size at which an organism reproduces.<span class="citation"><sup>[13‚Äì15]</sup></span> When exposed to a warmer environment, an organism may grow faster but mature younger at a smaller size, a pattern of phenotypic plasticity observed among some ectotherms<span class="citation"><sup>[16‚Äì18]</sup></span> and many endotherms.<span class="citation"><sup>[19,20]</sup></span> If a warmer environment causes an organism to reproduce when younger and smaller, the smaller parent should make fewer or smaller offspring. In addition, variation in body size may also depends on precipitation, which is a good predictor of primary production.<span class="citation"><sup>[21,22]</sup></span> Generally, abundant precipitation and radiation increase the rate of photosynthesis,<span class="citation"><sup>[23]</sup></span> which provides the energy required for growth and reproduction of organisms at higher trophic levels. If more productive areas enable organisms to consume more prey, one should expect these organisms to mature younger at smaller size; this indirect effect on maternal size should affect the size and number of offspring of the mother, as discussed earlier.
</p>
<p align="justify">
In addition to the effects of the environment, the foraging efficiency of an organism may profoundly influence its life history. The ability of an organism to gather and assimilate more food may promote follicular growth during the reproductive season; it may also increase energy stores to initiate reproduction, and it may reduce age at first reproduction. For example, female vipers that had good body condition early in vitellogenesis produced large litters.<span class="citation"><sup>[24]</sup></span> Similarly, vipers that gained more mass during follicular growth produced larger offspring. Early reproduction gives offspring sufficient time to mature in the same year that they hatched, which enables them to participate as adults in the subsequent breeding season.<span class="citation"><sup>[25]</sup></span>
</p>
<div id="current-state-of-knowledge" class="section level2 unnumbered">
<h2>Current state of knowledge</h2>
<p align="justify">
Although a covariation among environmental and maternal factors is expected to drive the evolution of the life history, most studies have focused on temperature as the main selective pressure. This bias has resulted in a wealth of conflicting results in the current literature as several comparative and species-specific studies have found no consistent trend. For instance, an analysis of over 100 studies,<span class="citation"><sup>[13]</sup></span> concluded that in more than 80% of the cases, decreased temperature leads to maturation later at a larger size. However, the variation among populations in natural environments paints a more complex picture. Given that environmental temperature generally increases with decreasing latitude, we should expect to observe a positive relationship between latitude and offspring size. Interestingly, a meta-analysis by<span class="citation"><sup>[26]</sup></span> revealed no general relationship between latitude and offspring size; relationships ranged from strongly negative to strongly positive. Moreover, the variation among natural populations likely reflects a combination of phenotypic plasticity and genetic divergence. A recent experiment by<span class="citation"><sup>[27]</sup></span> illustrates this point. They compared the reproductive traits of lizards from low or high elevation to that of lizards transplanted from low elevation to high elevation. Lizards from high elevation produced larger eggs than did lizards from low elevation; however, lizards transplanted to high elevation produced eggs of intermediate size. Thus, the difference in egg size between populations might partly stem from genetic factors, as well as environmental conditions during early development. Moreover, body size has diverse effects on reproductive traits, when analyzed independently of temperature. Larger females increase annual biomass production,<span class="citation"><sup>[28]</sup></span> produce more eggs,<span class="citation"><sup>[29]</sup></span> or both larger and more eggs.<span class="citation"><sup>[30]</sup></span> If these diverse patterns reflect diverse processes among taxa, a model focused on a single mechanism would be unable to explain variation in reproductive traits. Therefore, a unified approach to examine the combined effects of different environmental and maternal factors on the broad-scale variation in reproductive traits might provide the best insight.
</p>
<p align="justify">
Although experimental data on foraging efficiency are limited, comparative methods have been used to explore how foraging mode and life-history traits have evolved. These interspecific analyses have focused mostly on the reproductive effort of lizards. The earliest analysis of 22 species revealed an interesting pattern: sit-and-wait species had a greater reproductive effort than widely-foraging species.<span class="citation"><sup>[31]</sup></span> The authors suggested that widely-foraging species might have to carry fewer or smaller offspring, because moving long distances with a voluminous clutch compromises speed, which in turn decreases the chance of escaping a predator. A subsequent analysis of data for 50 species of lizards supports this result by testing a model in which predation risk increased with increasing reproductive effort.<span class="citation"><sup>[31]</sup></span> Roff (2002)<span class="citation"><sup>[1]</sup></span> extended support for this model in a comparative analysis of 130 species of lizards. However, none of these early analyses controlled for potential phylogenetic correlations that might generate spurious relationships between foraging mode and reproductive effort,<span class="citation"><sup>[32]</sup></span> especially because foraging mode varies more among families of lizards than within them. A more recent analysis, using phylogenetic comparative methods, failed to detect a significant relationship between foraging mode and reproductive effort.<span class="citation"><sup>[33]</sup></span> Thus, the long-standing prediction of a relationship between foraging mode and life-history traits is currently unsupported.
</p>
</p>
<p align="justify">
The first two chapters of this manuscript focus on a macroevolutionary approach to understand how variation in environmental conditions and maternal factors, including their foraging mode, affect the reproductive traits of lizards. The last two chapters rely on a microevolutionary approach to examine the evolution of foraging behavior in <em>Drosophila melanogaster</em>.
</p>
</div>
</div>
<div id="chapter-macroclimatic-and-maternal-effects-on-the-evolution-of-reproductive-traits-in-lizards" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Chapter: Macroclimatic and maternal effects on the evolution of reproductive traits in lizards</h1>
<div id="introduction" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> Introduction</h2>
<p align="justify">
Much of life-history theory rests on fundamental assumptions about constraints on the acquisition and allocation of energy to growth and reproduction. In general, the allocation of energy to reproduction depends on maternal size, which in turn depends on environmental factors experienced throughout the life of the mother (Figure <a href="#fig:fig1">1.1</a>).
</p>
<p align="justify">
Phylogenetic path analysis is an effective way to test hypotheses on the relationships among macroclimatic factors, maternal factors, and reproductive traits of lizards. Previously,<span class="citation"><sup>[34]</sup></span> combined path modeling, phylogenetic analysis, and information theory to infer the most likely relationships among environmental temperature, body size, and reproductive traits for 19 populations of sceloporine lizards. They found that temperature likely affects the number and size of offspring indirectly, via its effect on maternal body size. Since then, other researchers compiled published data for body sizes and reproductive traits of several species of lizards.<span class="citation"><sup>[28,35‚Äì38]</sup></span> Here, we use these data to evaluate competing hypotheses about the evolution of reproductive traits through phylogenetic path analyses (Figure <a href="#fig:fig1">1.1</a>). In doing so, we found that annual precipitation is the major driver of the life history. By using phylogenetic path analysis, we suggest a more general mechanism underlying the evolution of life-history traits of lizards.
</p>
</div>
<div id="materials-and-methods" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Materials and Methods</h2>
<p align="justify">
We used published estimates of life histories for 486 species of lizards, belonging to 34 families. We used the mean snout-vent length for hatchlings or neonates (mm), and adult females, as a measure of body size among species. Similarly, we averaged the lowest and highest reported means of clutch or litter sizes for each species (see<span class="citation"><sup>[36]</sup></span> for details). We used the length of hatchlings as an estimate of offspring size, rather than the mass of eggs, because the latter might reflect variation in water content rather than energy content in species of lizards that lay poorly calcified eggs.<span class="citation"><sup>[37,39]</sup></span> Moreover, hatchling size can be estimated for viviparous species as well as oviparous species.
</p>
<style>

p.caption {
  font-size: 0.9em;
  text-align: justify;

}

caption {

  font-size: 0.9em;
  text-align: justify;

}


</style>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig1"></span>
<img src="imgs/rev_hyp_bt.jpeg" alt="Path models depicting relationships among body temperature and life-history traits derived from optimality models. Abbreviation in the path diagrams are as follows: BT = body temperature; CS = clutch size; FS = maternal length; HS = hatchling/neonate length; PP = primary production; PR = annual precipitation." width="500" />
<p class="caption">
Figure 1.1: Path models depicting relationships among body temperature and life-history traits derived from optimality models. Abbreviation in the path diagrams are as follows: BT = body temperature; CS = clutch size; FS = maternal length; HS = hatchling/neonate length; PP = primary production; PR = annual precipitation.
</p>
</div>
<p align="justify">
Because the range of nearly all reptiles have been recently mapped,<span class="citation"><sup>[40]</sup></span> we could obtain data on annual air temperature and annual precipitation across the range of each species. These climatic variables derived from the monthly values of temperature and rainfall for global land areas at a resolution of 30 s.<span class="citation"><sup>[41]</sup></span> Similarly, we obtained data on log-transformed values of net primary production‚Äîthe net amount of solar energy converted to plant organic matter through photosynthesis measured in units of elemental carbon. This measure represents the primary source of trophic energy for the world‚Äôs ecosystems.<span class="citation"><sup>[42]</sup></span> We used polygonal range maps representing species extent of occurrence to run a zonal statistical algorithm in the software QGIS version 3.22.2-Bia≈Çowie≈ºa; this algorithm enabled us to extract the means of climatic data across pixels of a map for each species. In addition to extracting climatic data, we also gathered published records of body temperature measured in the field for 320 species. These data consist of an average of the minimal and maximal temperatures reported in the literature.<span class="citation"><sup>[36]</sup></span>
</p>
</div>
<div id="statistical-analyses" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> Statistical analyses</h2>
<p align="justify">
We tested our hypotheses (see Figure <a href="#fig:fig1">1.1</a>) by performing phylogenetic path analysis of data of body temperature, and environmental temperature. Examining the explanatory power of competing models involving different relationships among variables enabled us to identify the most plausible hypothesis given the available data, and thus infer the relative importance of mechanisms described by optimality models. These analyses were performed with the <em>phylopath</em> library of the Statistical Software R version 4.0.5,<span class="citation"><sup>[43,44]</sup></span> and a recent time-calibrated phylogeny for squamate taxa.<span class="citation"><sup>[45]</sup></span> We calculated each model‚Äôs goodness of fit with the Fisher‚Äôs C statistic,<span class="citation"><sup>[46]</sup></span> obtained as follows:
</p>
<p><span class="math display">\[C = -2\sum_{i=1}^{k}ln(p_{i}) \]</span></p>
<p align="justify">
where <span class="math inline">\(k\)</span> is the number of conditional independencies in the minimal set and <span class="math inline">\(p_{i}\)</span> is the null probability associated with each of the predicted independence claims tested. A conditional independency specifies the list of pairs of variables that are statistically independent conditioning on a set of other variables in the causal model. For example, a conditional independency that supports one of our hypotheses relates body temperature (BT), maternal size (FS) and clutch size (CS; see Figure <a href="#fig:fig1">1.1</a>b). In this example, clutch size is d-separated from body temperature, which means that there is not an arrow linking these variables (or ‚Äúvertices‚Äù) in the directed acyclic graphic. Maternal size is considered the causal ‚Äúparent‚Äù of both body temperature and clutch size, because it is the variable directly linked with both vertices (BT and CS). It is possible now to translate these d-separation statements to statistical linear models in which we test the independence of clutch size and temperature, conditioning on their parent, maternal size. The C statistic is a maximum likelihood estimate that follows a <span class="math inline">\(x^2\)</span> distribution with degrees of freedom <span class="math inline">\(df=2k\)</span>. Therefore, it provides a convenient statistic for testing the goodness of fit of the whole path model.<span class="citation"><sup>[47]</sup></span> The path model does not provide a good fit to the data if the <span class="math inline">\(p\)</span> value of the <span class="math inline">\(C\)</span> statistic is below the alpha value (0.05). We compared models using a modified version of the Akaike Information Criterion (<span class="math inline">\(AIC\)</span>) known as the <span class="math inline">\(C\)</span> statistic Information Criterion (<span class="math inline">\(CIC\)</span>), proposed by.<span class="citation"><sup>[48]</sup></span> Here, we calculated the <span class="math inline">\(CIC_{c}\)</span>, which is the equivalent of <span class="math inline">\(CIC\)</span> with a correction for small sample sizes, as follows:
</p>
<p><span class="math display">\[CIC_{c}=C+2q\frac{n}{(n-1-q)}\]</span></p>
<p align="justify">
where <span class="math inline">\(C\)</span> is the maximum likelihood of the particular model, <span class="math inline">\(q\)</span> is the number of parameters estimated in the path model, and <span class="math inline">\(n\)</span> is the sample size (number of species). The most likely model has the lowest <span class="math inline">\(CIC_{c}\)</span> value.<span class="citation"><sup>[49]</sup></span>
</p>
<p align="justify">
Our data analyses were entirely based on full model averaging, by which one calculates a likelihood-weighted average of parameters among all models ranked. Model averaging causes the path coefficients that do not occur in all models to shrink toward zero.<span class="citation"><sup>[43]</sup></span> We reported the conditional independencies that supported the averaged model. We also reported the p-values of each independence statements, and an estimate of the Pagel‚Äôs <span class="math inline">\(\lambda\)</span> to account for phylogenetic non-independence of each statement. The parameter <span class="math inline">\(\lambda\)</span> can vary between 0 and 1; a value of 0 for a conditional independency indicates that the relationship between the life histories involved is not constrained by the phylogeny, whereas a value of 1 indicates the converse.
</p>
</div>
<div id="results" class="section level2" number="1.4">
<h2><span class="header-section-number">1.4</span> Results</h2>
<p align="justify">
Phylogenetic path analyses revealed that annual precipitation directly and indirectly affected the evolution of the reproductive traits in lizards (Figure <a href="#fig:fig2">1.2</a>). The direct effect stems from its negative relationship with hatchling size, while the indirect effect is mediated by its effect on maternal size. These analyses also indicate that body temperature had little or no effect on the evolution of reproductive tactics, either directly or indirectly. We drew this conclusion from the coefficients derived from full model averaging, which indicated negligible effects of body temperature on maternal size and hatchling size (Figure <a href="#fig:fig2">1.2</a>a). Importantly, we found a similar pattern when using the mean environmental temperatures across the species range as an independent variable in our analysis (Figure <a href="#fig:fig2">1.2</a>b).
</p>
<p align="justify">
Our analyses also revealed that maternal size greatly determined the evolution of hatchling size and clutch size. Specifically, larger females simultaneously produce larger offspring and more offspring (Figure <a href="#fig:fig2">1.2</a>). By accounting for maternal size in our phylogenetic path analysis, we observed the expected negative relationship between hatchling size and clutch size. Likely, larger females produce larger clutches of larger offspring by having more surplus energy to allocate to reproduction (Figure <a href="#fig:fig3">1.3</a>).
</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig2"></span>
<img src="imgs/rev_pannel.jpeg" alt="Best-fit models of the evolution of reproductive traits considering body temperature (a) and environmental temperature (b) as independent variable. Thicker arrows indicate stronger effects. Values in parentheses denote the standard errors of the path coefficients." width="500" />
<p class="caption">
Figure 1.2: Best-fit models of the evolution of reproductive traits considering body temperature (a) and environmental temperature (b) as independent variable. Thicker arrows indicate stronger effects. Values in parentheses denote the standard errors of the path coefficients.
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig3"></span>
<img src="imgs/tree4.jpeg" alt="Evolution of the reproductive effort among 669 species of lizards. The dataset used to perform path analyses was reduced to 486 species because we could not extract data of climatic variables for all species present in the phylogeny." width="500" />
<p class="caption">
Figure 1.3: Evolution of the reproductive effort among 669 species of lizards. The dataset used to perform path analyses was reduced to 486 species because we could not extract data of climatic variables for all species present in the phylogeny.
</p>
</div>
</div>
<div id="discussion" class="section level2" number="1.5">
<h2><span class="header-section-number">1.5</span> Discussion</h2>
<p align="justify">
A model describing a direct and an indirect effect of precipitation on the evolution of reproductive traits was strongly supported (see Figure <a href="#fig:fig1">1.1</a>g). On one hand, we found a negative relationship between precipitation and hatchling size. On the other hand, we found a positive effect of precipitation on maternal size. Both relationships might reflect selective pressures on body size associated with food availability. Greater precipitation often results in greater primary production, which translates to greater food abundance for lizards.<span class="citation"><sup>[21,22]</sup></span> In a more productive environment, a juvenile lizard could mature younger at smaller size. Additionally, given the tradeoff between the size and number of offspring, a mature lizard should produce more, smaller offspring in an environment that favors growth. Although the averaged model indicates no strong direct effect of primary production on maternal size (Figure <a href="#fig:fig2">1.2</a>a), our analysis revealed that a model including this path should also be considered important based on the <span class="math inline">\(\Delta CIC_{c}\)</span> (Figure <a href="#fig:fig4">1.4</a>). Alternatively, we consider other mechanisms by which precipitation can directly influence body size. The water-conservation hypothesis provides one such mechanism. This hypothesis predicts stronger selection for large size in dry environments, given that the size-specific rate of water loss decreases with increasing size.<span class="citation"><sup>[50‚Äì53]</sup></span>
</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig4"></span>
<img src="imgs/mod_sel.jpeg" alt="Relative importance of the causal models describing the evolution of reproductive traits in lizards." width="500" />
<p class="caption">
Figure 1.4: Relative importance of the causal models describing the evolution of reproductive traits in lizards.
</p>
</div>
<p align="justify">
The absence of a thermal effect on the evolution of reproductive tactics might partially be explained by thermoregulation. Many species of lizards use behavior and physiology to buffer variation in environmental temperature.<span class="citation"><sup>[54]</sup></span> This buffering ability reduces selective pressures on thermal physiology, slowing evolutionary divergence over space and time.<span class="citation"><sup>[54‚Äì56]</sup></span> The conservation of thermal tolerance within species has been widely recognized,<span class="citation"><sup>[56,57]</sup></span> but this phenomenon is also evident among some species of lizards.<span class="citation"><sup>[58,59]</sup></span> Body temperature constrains life histories because it exerts an important influence on the acquisition of energy required for growth and reproduction. Normally, the detection, capture, ingestion, digestion, and assimilation of food are processes that determine the acquisition of energy. These processes perform well over an optimal range of body temperatures and perform poorly at temperatures outside this optimum.<span class="citation"><sup>[60]</sup></span> Whether a lizard‚Äôs optimal temperature for performance remains similar in space and time depends on the effectiveness of thermoregulation. A thermoregulating lizard might bask in open areas more often at high altitudes or latitudes, but it might use shaded areas more often at low altitudes or latitudes.<span class="citation"><sup>[61,62]</sup></span> Consequently, the optimal body temperatures of species from different altitudes or latitudes might not change drastically.<span class="citation"><sup>[54]</sup></span> If this behavior enabled some lizards to maintain their body temperature close to the thermal optimum regardless of the climate, thermoregulation would have slowed the divergence of life-history traits among species. Because nocturnal lizards have fewer opportunities to behaviorally thermoregulate than diurnal lizards do, we expect temperature to have a greater effect on the reproductive tactics of these lizards.
</p>
<p align="justify">
Additionally, local adaptation of thermal physiology among species might reduce the impact of environmental temperature on the life history of lizards. If the preferred body temperatures of species have diverged to track the environmental temperatures of their habitats, natural selection would favor a subsequent shift in the thermal optima for performance.<span class="citation"><sup>[63]</sup></span> This coadaptation of thermoregulatory behavior and thermal physiology would enable each species to perform best at the body temperature commonly experienced in its environment. In lizards,<span class="citation"><sup>[64]</sup></span> found a significant correlation between the preferred temperature and the thermal optima for sprint speed among 13 species of lacertids. Similarly,<span class="citation"><sup>[65]</sup></span> found that thermal optima for sprinting correlated strongly with field body temperature among 19 species of iguanid lizards. Thus, the coadaptation of thermoregulatory behavior and thermal physiology seems a likely factor contributing to the patterns of interspecific variation that we observed in general among lizards.
</p>
<p align="justify">
Our study indicates that precipitation is the most likely driver of the evolution of reproductive traits in lizards. Although biologists have intensely debated the mechanisms by which temperature affects body size,<span class="citation"><sup>[15,66,67]</sup></span> we found no consistent relationship between temperature and body size among species of lizards. Interestingly, we now know that lizards with larger amounts of reproductive resources produce both larger hatchlings and larger clutches. Although we do not know the selective pressures responsible for this pattern, theoretical models tell us that parental care or intraspecific competition could play a role.<span class="citation"><sup>[6]</sup></span> In lizards, we lack experimental evidence that larger females provide better parental care. We also do not know the relationship between offspring size and competitive ability for most species. Therefore, empirical evidence of this nature would clarify whether these mechanisms play any role in the evolution of life histories in lizards. For longer-lived species such as lizards, experimental studies of life-history evolution have lagged behind comparative studies. Still, phylogenetic path analysis has enabled us to disentangle some evolutionary relationships among life histories. This analysis sharpens our focus on mechanisms underlying major evolutionary patterns, stimulating the development of a more general life-history theory.
</p>
</div>
</div>
<div id="chapter-the-correlated-evolution-of-foraging-mode-and-reproductive-effort-in-lizards" class="section level1" number="2">
<h1><span class="header-section-number">2</span> Chapter: The correlated evolution of foraging mode and reproductive effort in lizards</h1>
<div id="introduction-1" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Introduction</h2>
<p align="justify">
Life-history theory suggests that the optimal reproductive effort of an organism is affected by factors such as energy acquisition and predation risk. The observation that some organisms actively search for their prey and others ambush them creates the expectation of different energy needs and predation risk associated with each foraging behavior, the so-called ‚Äúforaging-mode paradigm.‚Äù In vertebrates, the foraging behaviors lie along a continuum, ranging from the energetically demanding strategy of searching for prey to the energetically conservative strategy of ambushing prey.<span class="citation"><sup>[68,69]</sup></span> During the past 50 years, ecologists have developed a set of hypotheses about how an organism‚Äôs foraging mode relates to its life history.<span class="citation"><sup>[11,31,70,71]</sup></span> These relationships stem from two major assumptions (Figure <a href="#fig:fig5">2.1</a>). First, a widely-foraging lizard spends more energy while foraging than a sit-and-wait lizard, but might also consume enough food during an activity season to have more surplus energy.<span class="citation"><sup>[72‚Äì77]</sup></span> However, a sit-and-wait lizard might have a more diverse diet, because it encounters prey less frequently than a widely-foraging lizard does.<span class="citation"><sup>[78]</sup></span> Second, a widely-foraging lizard could suffer a greater risk of predation when movements are conspicuous to predators.<span class="citation"><sup>[79]</sup></span> Risk of predation might also affect the evolution of reproductive effort, because carrying a greater mass of offspring reduces a female‚Äôs ability to evade a predator.<span class="citation"><sup>[80]</sup></span> Both a greater energy supply and a greater mortality risk would select for a genotype that matures at an earlier age and allocates more energy to reproduction, manifested as more or larger offspring.<span class="citation"><sup>[24,81]</sup></span> Finally, the load associated with food consumption may compromise the speed of an animal, increasing its vulnerability to predators.<span class="citation"><sup>[82,83]</sup></span> In this model, a suite of traits associated with foraging mode would coadapt to the spatiotemporal distributions of prey and predators.
</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig5"></span>
<img src="imgs/Figure1.jpeg" alt="A conceptual model depicting putative relationships among foraging behavior, energetics, predation risk, and reproductive effort. The predicted relationships were derived from theoretical models of life-history evolution." width="500" />
<p class="caption">
Figure 2.1: A conceptual model depicting putative relationships among foraging behavior, energetics, predation risk, and reproductive effort. The predicted relationships were derived from theoretical models of life-history evolution.
</p>
</div>
<p align="justify">
Despite the wealth of conjecture, these hypothetical relationships among foraging mode and other traits have been assessed in only a handful of cases. Ideally, one would isolate each relationship and conduct experiments to quantify the evolution of traits in a controlled environment.<span class="citation"><sup>[84]</sup></span> However, such data are difficult to gather for many species and will probably remain rare. For instance, limited circumstantial evidence exists for the putative relationship between foraging mode and predation risk. Stomach contents of vipers suggested that widely-foraging species of lizards are more vulnerable to predators than sedentary species,<span class="citation"><sup>[72]</sup></span> but a number of confounding factors can explain this observation as well as foraging mode can. Alternatively, researchers have evaluated the relationship between mortality and life-history traits with no emphasis on foraging behavior. For example, experimental evolution with guppies and fruit flies revealed that genotypes that evolved in risky environments developed more rapidly, matured at a smaller size, and reproduced earlier in life than did genotypes that evolved in safe environments.<span class="citation"><sup>[2,85]</sup></span> From these results, we expect that a greater risk of predation among widely-foraging species selects for greater reproductive effort; however, given the scarce evidence, we cannot conclude whether foraging mode generally affects predation risk.
<p align="justify">
Here present the first evidence that the evolution of widely-foraging in lizards was associated with the evolution of greater reproductive effort, specifically among large-bodied species of lizards. This evidence comes from a comparative analysis of foraging mode and reproductive effort comprising 485 species of lizards from 32 families. In this analysis, we inferred the evolutionary history of foraging modes, complementing past reconstructions of ancestral states.<span class="citation"><sup>[86]</sup></span> In contrast to previous analyses, our study partially supports the prediction of theoretical models of the optimal reproductive effort, paving the way for ecologists to test mechanistic hypotheses at the intraspecific level.
</p>
</div>
<div id="materials-and-methods-1" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Materials and Methods</h2>
<p align="justify">
We used published estimates of life history and foraging behavior for 485 species of lizards grouped in 32 families, excluding amphisbaenians and snakes. These data represent a subset of those assembled by ecologists from primary and secondary literatures.<span class="citation"><sup>[36]</sup></span> We defined reproductive effort as the mean product of offspring mass and offspring number. Accordingly, we computed the product of the mean scaled mass index for hatchlings or neonates, <span class="math inline">\(\hat M(g)\)</span>, and the mean clutch or litter size for each species of lizards. This product was interpreted in terms of reproductive effort after adjusting for the mass of the parent (i.e., the mean <span class="math inline">\(\hat M(g)\)</span> of adult females). The scaled mass index standardizes body mass at a given fixed value of a linear body measurement, based on the scaling relationship between mass and length.<span class="citation"><sup>[87]</sup></span> We first estimated hatchling mass and maternal mass from values of snout-vent-length (SVL) using published allometric equations.<span class="citation"><sup>[36,37,88]</sup></span> Then, we computed <span class="math inline">\(\hat M\)</span> as follows:
</p>
<p><span class="math display">\[\hat M=M_i(\frac{L_0}{L_i})^b_{SMA} \]</span></p>
<p align="justify">
where <span class="math inline">\(M_i\)</span> and <span class="math inline">\(L_i\)</span> are the raw body mass and the linear body measurement of individual <span class="math inline">\(i\)</span>, respectively; <span class="math inline">\(b_{SMA}\)</span> is the scaling exponent estimated by the standardized major-axis regression of <span class="math inline">\(ln(M)\)</span> on <span class="math inline">\(ln(L)\)</span>; <span class="math inline">\(L_0\)</span> is an arbitrary value of <span class="math inline">\(L\)</span> (e.g., the arithmetic mean of SVL values for the study species); and <span class="math inline">\(\hat M_i\)</span> is the predicted body mass for individual <span class="math inline">\(i\)</span> when the linear body measure is standardized to <span class="math inline">\(L_0\)</span>. The scaled mass index performs best as a predictor of variation of energy stores, such as fat as well as other body components.<span class="citation"><sup>[87]</sup></span> Given that we could not find clutch frequencies and ages of first reproduction for all of the widely-foraging and sit-and-wait species in this study, our analysis focused on reproductive effort in a single event rather than the lifetime reproductive effort, which corresponds to the total production of offspring over the lifetime of a female.
</p>
<p align="justify">
Among species of lizards, widely foraging females and sit-and-wait females have been compared mostly in terms of their relative clutch mass. Relative clutch mass accounts for variation in energy availability among individuals by computing the ratio of reproductive mass to maternal mass, where mass is a proxy for energy. Although relative clutch mass is commonly used as measure of reproductive effort, regressing this index on female body mass for further analyses might be problematic, because ratios of random numbers regressed against their denominator will yield spurious correlations.<span class="citation"><sup>[89,90]</sup></span> By regressing the mean product of offspring mass and offspring number on maternal mass, the slope of the linear relationship can be interpreted as reproductive effort‚Äîproportion of mass allocated to reproduction at a given maternal mass‚Äîwhich enabled us to avoid statistical issues associated with the analysis of ratios.
</p>
<p align="justify">
We defined the foraging mode of a species based on whether it has been reported as ambushing prey (sit-and-wait forager), searching for prey (widely foraging), or using a mixed strategy (mixed). Although this categorization seems somewhat artificial, most species of lizards clearly belong to one of these categories.<span class="citation"><sup>[69,72]</sup></span> We focused our analyses only on carnivores, because herbivores do not fit into the classical paradigm of foraging modes.<span class="citation"><sup>[91]</sup></span>
</p>
</div>
<div id="statistical-analyses-1" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Statistical analyses</h2>
<p align="justify">
To determine the appropriate model of evolution, we used a set of continuous-time, discrete-state Markov-chain models to sample the character histories from their posterior probability distribution,<span class="citation"><sup>[92]</sup></span> and a time-calibrated phylogeny of squamate reptiles.<span class="citation"><sup>[45]</sup></span> We rooted the tree with the tuatara (<em>Sphenodon punctatus</em>) as the outgroup for our study taxa. Based on previous analyses, we coded this outgroup as a sit-and-wait species.<span class="citation"><sup>[91,93]</sup></span> However, we also performed the analysis with no outgroup to evaluate the consistency of our estimates. Additionally, we estimated phylogenetic signal (character dispersion on a phylogeny) using Fritz and Purvis‚Äô <span class="math inline">\(D \ test\)</span>, available through the function <em>phylo.d</em> in the ‚Äúcaper‚Äù package of R, version 1.0.1.<span class="citation"><sup>[44,94,95]</sup></span>
</p>
<p align="justify">
We used Phylogenetic Generalized Least Squares (PGLS) to model the relationship among maternal mass, foraging mode, and reproductive output. PGLS models enabled us to account for non-independence of the data.<span class="citation"><sup>[32,96]</sup></span> To do so, we used the <em>gls</em> function from the ‚Äúnlme‚Äù package of R, version 3.1.153.<span class="citation"><sup>[44,97]</sup></span> We included maternal mass and foraging mode as independent variables and the mean product of hatchling mass and clutch size as the dependent variable. Then we fitted multiple models assuming that the species trait values evolved via a Brownian motion model, an Ornstein-Uhlenbeck motion model, and a Pagel‚Äôs lambda model using the R package ‚Äúape‚Äù version 5.6.2. To evaluate the models‚Äô goodness of fit, we used information-theoretic criteria such as <span class="math inline">\(AIC_c\)</span>. We ranked candidate models accordingly and selected the most likely one (lowest value of AICc) for inferences. To control for variation in maternal size among species, we examined models in which maternal mass was standardized (<span class="math inline">\(\hat M\)</span>). This analysis enabled us to deal with outliers, thus increasing the robustness of our conclusions.
</p>
</div>
<div id="results-1" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> Results</h2>
<p align="justify">
The evolutionary transitions in foraging mode among species were best described by a model in which rates of evolution were equal among foraging modes, referred to as the equal-rates model (ER). According to the posterior probabilities from stochastic mapping, sit-and-wait foraging is the most likely ancestral state of all lacertilians (Figure <a href="#fig:fig6">2.2</a>). We drew this conclusion from the ancestral state reconstruction analysis including the outgroup (i.e.¬†the tuatara) and with no outgroup. Sit-and-wait foraging evolved in two major clades of lizards; near the root of the tree, we found strong evidence suggesting that the ancestor of the basal gekkotans was likely a sit-and-wait predator. Likewise, the ancestor of the more derived iguanians was likely a sit-and-wait predator. During the Jurassic and middle Cretaceous, major transitions from sit-and-wait foraging to widely foraging seemed to have occurred in the ancestors of Scincoidea, Lacertoidea, and Anguimorpha. However, a major reverse transition from widely foraging to sit-and-wait foraging has also occurred during the Jurassic, in the ancestor of iguanians. The <span class="math inline">\(D\)</span> test for phylogenetic signal indicated that foraging mode is phylogenetically conserved among lizards (<span class="math inline">\(D=-0.14, \ p[D&lt;1]=0, \ p[D&gt;0]=0.80\)</span>). Accordingly, closely related species tend to exhibit the same foraging mode, but this similarity decreases as phylogenetic distant increases.
</p>
<p align="justify">
The evolution of reproductive effort in lizards was driven by an interaction between maternal mass and foraging mode (Table <a href="#tab:tab1">2.1</a>). The information-theoretic approach revealed that a model including this interaction was strongly supported (Table <a href="#tab:tab2">2.2</a>). Widely foraging was associated with the evolution of greater reproductive effort than was sit-and-wait foraging, specifically in large-bodied species of lizards (Figure <a href="#fig:fig7">2.3</a>a). Yet, this difference seemed small when hatchling mass and maternal mass were standardized (Figure <a href="#fig:fig7">2.3</a>b). A model in which mass was standardized, <span class="math inline">\(\hat M\)</span>, suggested that species adopting a mixed-foraging strategy evolved the greatest reproductive effort (Figure <a href="#fig:fig7">2.3</a>b).
</p>
<table class="table" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:tab1">Table 2.1: </span>Analysis of deviance table (Type III tests) for the most likely
model, based on the ranking of <span class="math inline">\(AIC_c\)</span> for potential candidate models.
</caption>
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:right;">
Df
</th>
<th style="text-align:right;">
Chisq
</th>
<th style="text-align:right;">
p.value
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
(Intercept)
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.005
</td>
<td style="text-align:right;">
0.942
</td>
</tr>
<tr>
<td style="text-align:left;">
M_females
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
4.761
</td>
<td style="text-align:right;">
0.029
</td>
</tr>
<tr>
<td style="text-align:left;">
foraging.mode
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
10.372
</td>
<td style="text-align:right;">
0.006
</td>
</tr>
<tr>
<td style="text-align:left;">
M_females:foraging.mode
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
18.105
</td>
<td style="text-align:right;">
0.000
</td>
</tr>
</tbody>
</table>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig6"></span>
<img src="imgs/Figure2.jpeg" alt="Random sample of stochastic character maps depicting the evolution of foraging mode in 485 species of lizards. Bars at the tips of the phylogeny represent log-transformed values of reproductive effort for all lizards, but not the outgroup, *Sphenodon punctatus*. Pie charts on internal nodes represent posterior probability estimates resulting from 1,000 simulations of the character histories. Major clades are enumerated as follows: 1) Gekkota, 2) Scincoidea, 3) Lacertoidea, 4) Anguimorpha, 5) Toxicofera, and 6) Iguania. Lizard photos by Mark O‚ÄôShea." width="100%" />
<p class="caption">
Figure 2.2: Random sample of stochastic character maps depicting the evolution of foraging mode in 485 species of lizards. Bars at the tips of the phylogeny represent log-transformed values of reproductive effort for all lizards, but not the outgroup, <em>Sphenodon punctatus</em>. Pie charts on internal nodes represent posterior probability estimates resulting from 1,000 simulations of the character histories. Major clades are enumerated as follows: 1) Gekkota, 2) Scincoidea, 3) Lacertoidea, 4) Anguimorpha, 5) Toxicofera, and 6) Iguania. Lizard photos by Mark O‚ÄôShea.
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig7"></span>
<img src="imgs/Figure3.jpeg" alt="Effects of maternal mass and foraging mode on the evolution of reproductive effort of lizards, as determined by phylogenetic generalized least squares analysis. (a) Difference in reproductive effort among foraging modes assuming body mass as a proxy for energy. (b) Difference in reproductive effort among foraging modes assuming the scaled mass index as a proxy for energy." width="500" />
<p class="caption">
Figure 2.3: Effects of maternal mass and foraging mode on the evolution of reproductive effort of lizards, as determined by phylogenetic generalized least squares analysis. (a) Difference in reproductive effort among foraging modes assuming body mass as a proxy for energy. (b) Difference in reproductive effort among foraging modes assuming the scaled mass index as a proxy for energy.
</p>
</div>
<p><br></p>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<caption>
<span id="tab:tab2">Table 2.2: </span>Contrast of parameters estimated by the most likely model, according to the information-theoretic criteria of selection.
</caption>
<thead>
<tr>
<th style="text-align:left;">
Model
</th>
<th style="text-align:right;">
K
</th>
<th style="text-align:right;">
logLik
</th>
<th style="text-align:right;">
AICc
</th>
<th style="text-align:right;">
delta
</th>
<th style="text-align:right;">
weight
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
rep_out ~ M_females * foraging.mode
</td>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
-1377.419
</td>
<td style="text-align:right;">
2771.141
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.999
</td>
</tr>
<tr>
<td style="text-align:left;">
rep_out ~ M_females + foraging.mode
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
-1386.401
</td>
<td style="text-align:right;">
2784.977
</td>
<td style="text-align:right;">
13.837
</td>
<td style="text-align:right;">
0.001
</td>
</tr>
<tr>
<td style="text-align:left;">
rep_out ~ M_females * foraging.mode
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
-1434.910
</td>
<td style="text-align:right;">
2884.055
</td>
<td style="text-align:right;">
112.915
</td>
<td style="text-align:right;">
0.000
</td>
</tr>
<tr>
<td style="text-align:left;">
rep_out ~ M_females + foraging.mode
</td>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
-1440.896
</td>
<td style="text-align:right;">
2891.918
</td>
<td style="text-align:right;">
120.777
</td>
<td style="text-align:right;">
0.000
</td>
</tr>
</tbody>
</table>
</div>
<div id="discussion-1" class="section level2" number="2.5">
<h2><span class="header-section-number">2.5</span> Discussion</h2>
<p align="justify">
Our analysis provides evidence that species of widely-foraging lizards have evolved greater reproductive effort, but only in heavier species (see Figure <a href="#fig:fig7">2.3</a>a). Presumably, this pattern stems from the ability of widely-foraging species to harvest and assimilate more resources than sit-and-wait species can. In the field, a widely-foraging lizard spends 32% more daily energy than a sit-and-wait lizard, but this extra energy expenditure is probably paid off with greater daily food consumption.<span class="citation"><sup>[98]</sup></span> The ability of widely-foraging lizards to forage more efficiently should reduce their risk of predation. Rapid growth by widely-foraging lizards might also cause them to outgrow the gape limitations of predators.<span class="citation"><sup>[99,100]</sup></span> Therefore, if body size is critical to outperform predators and outcompete conspecifics, the optimal pattern may be to grow to the maximal size that leads to the greatest reproductive effort.
</p>
<p align="justify">
Our study presents the first evidence that the early shift in foraging mode‚Äîfrom sit-and-wait foraging to widely foraging‚Äîin the evolutionary history of lacertilians was likely accompanied by the evolution of a greater reproductive effort, specifically in large-bodied species of lizards (Figure <a href="#fig:fig6">2.2</a>). Yet, much variation in reproductive effort exits among species with the same foraging mode, such that ecologists must consider other factors that influence reproduction. Importantly, our study captured the effects of foraging plasticity on the reproductive effort of lizards. For instance, lizards that adopt a mixed-foraging strategy produced the greatest reproductive effort. Hopefully, our findings encourage others to investigate how foraging efficiency and predation risk associated with foraging modes influence the evolution of reproductive traits.
</p>
</div>
</div>
<div id="chapter-foraging-efficiency-of-rover-and-sitter-larvae-of-drosophila-melanogaster" class="section level1" number="3">
<h1><span class="header-section-number">3</span> Chapter: Foraging efficiency of rover and sitter larvae of <em>Drosophila melanogaster</em></h1>
<div id="introduction-2" class="section level2" number="3.1">
<h2><span class="header-section-number">3.1</span> Introduction</h2>
<p align="justify">
In the previous chapter, we presented the first evidence that widely foraging species of lizards actually have greater reproductive effort than sit-and wait species, which we attributed to a greater rates of energy acquisition and mortality among mobile foragers. Thus, our work refutes the traditional view, calling into question long-standing predictions of the foraging-mode paradigm. In this chapter of my dissertation, I will pursue this area of research with an experimental approach involving the larvae of the fruit fly <em>Drosophila melanogaster</em>.
</p>
<p align="justify">
Currently, analyses of the efficiency of foraging behaviors stem from measures of a few species in short periods of a season. Studies of this nature in laboratory-controlled conditions are still rare in the current literature, revealing the need for long-term studies to help us understand the evolution of foraging behavior and its connection to the life history. In <em>Drosophila melanogaster</em>, a foraging gene (<em>for</em>) is a well-established example of a gene with major effects on foraging behavior.<span class="citation"><sup>[101]</sup></span> This gene is best known for underlying the foraging behavior strategies of rover (‚Äúactive foragers‚Äù) and sitter (‚Äúpassive foragers‚Äù) foraging larvae, making this organism a good study system to test the relationship between foraging behavior and energy acquisition.
</p>
<p align="justify">
The foraging efficiency of the <em>Drosophila</em> larvae can be examined based on a model of optimal feeding strategies.<span class="citation"><sup>[102]</sup></span> This model suggests that the energy accumulated from foraging (<span class="math inline">\(m\)</span>) increases monotonically towards an asymptote (Figure <a href="#fig:fig8">3.1</a>). If the total energy accumulated from foraging scales proportionally with body size, an asymptotic size then suggests that foraging for twice as long would not result in twice the optimal size. For example, if two larvae differ in body size, the smaller one would reach its asymptotic size at a lower value of <span class="math inline">\(m\)</span> (Figure <a href="#fig:fig8">3.1</a>a). While the asymptote represents the optimal body size that an organism would reach, the ratio of the energy accumulated for growth to the time spent foraging (<span class="math inline">\(\frac{m}{t}\)</span>) corresponds to its foraging efficiency. In some organisms, this ratio likely depends on their foraging mode, as well as on the abundance and distribution of food. Accordingly, if a rover larva has a higher foraging efficiency, it might grow faster and reach maturity at a smaller size than would a sitter larva that is less efficient at foraging (Figure <a href="#fig:fig8">3.1</a>b).
</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig8"></span>
<img src="index_files/figure-html/fig8-1.png" alt="A theoretical model relating the cumulative energy gain $m$, as a function of time spent foraging, $t$. (a) Larger larvae reach their maximum capacity at a higher value of $m$ than small larvae. (b) Rovers that are smaller but more efficient foragers may grow faster at a smaller size than larger and less efficient sitter larvae. $t_{r}$ and $t_{s}$ in (b) represent the optimal foraging time of rover and sitter, respectively." width="2100" />
<p class="caption">
Figure 3.1: A theoretical model relating the cumulative energy gain <span class="math inline">\(m\)</span>, as a function of time spent foraging, <span class="math inline">\(t\)</span>. (a) Larger larvae reach their maximum capacity at a higher value of <span class="math inline">\(m\)</span> than small larvae. (b) Rovers that are smaller but more efficient foragers may grow faster at a smaller size than larger and less efficient sitter larvae. <span class="math inline">\(t_{r}\)</span> and <span class="math inline">\(t_{s}\)</span> in (b) represent the optimal foraging time of rover and sitter, respectively.
</p>
</div>
</div>
<div id="experimental-design" class="section level2" number="3.2">
<h2><span class="header-section-number">3.2</span> Experimental design</h2>
<p align="justify">
The rover (<span class="math inline">\(for^R\)</span>) and sitter (<span class="math inline">\(for^S\)</span>) strains that we will use in our study have been provided by the Sokolowski lab at the University of Toronto, Canada. Flies of both strains will be separately reared on a mixture of cornmeal, corn syrup, and yeast-agar culture bottles incubated at 25<span class="math inline">\(^\circ\)</span>C and 60% relative humidity in a 12:12 h LD cycle. To set up the parental populations of each strain, we will collect 100-150 females and 50-75 males and age them for 5 days for optimal fecundity.<span class="citation"><sup>[103]</sup></span> We will allow them to mate overnight in a 6 oz culture bottle with standard food. The flies will be transferred to a culture bottle oriented upside down and sealed up with a grape plate at the bottom. By doing this, we will allow the adults to lay eggs on the grape plate. We will discard the first grape plate after 24 h and place a new grape plate in the bottle opening. This will be the grape plate from which larvae will be collected the next day.
</p>
<p align="justify">
The experimental set up will begin with a petri dish covered with a thin layer of agar matrix. The agar layer should be thin enough to enable us to observe a path that a moving larva would leave as it forages. Because the larvae actually feed on yeast, we will add small drops of yeast paste on the agar matrix in such a way that some petri dish will have drops of yeast uniformly distributed (Figure <a href="#fig:fig9">3.2</a>), and others will have the drops distributed in patches (Figure <a href="#fig:fig10">3.3</a>). Manipulating the distribution of yeast on the petri dish will enable us to examine the foraging efficiency of a larva when the distribution of food varies in the environment they are placed on. We will cover the test dish with a transparent lid for the larvae to remain visible during the duration of the experiments.
</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig9"></span>
<img src="index_files/figure-html/fig9-1.png" alt="Schematic representation of an arena with a hypothetical uniform distribution of resources." width="80%" />
<p class="caption">
Figure 3.2: Schematic representation of an arena with a hypothetical uniform distribution of resources.
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig10"></span>
<img src="index_files/figure-html/fig10-1.png" alt="Schematic represention of an arena with a multi-patch distribution of resources." width="65%" />
<p class="caption">
Figure 3.3: Schematic represention of an arena with a multi-patch distribution of resources.
</p>
</div>
<p align="justify">
Before the beginning of the experiments, a third instar larva will be first weighed on a precision scale and then it will be immediately transferred to the center of the test dish using a paintbrush. After covering the test dish with the transparent lid, we will record the foraging behavior of the larva using a camera placed 30 cm above the test dish. The recording will be stopped after 24 h, and the larva will be weighed once again. The difference between the final mass and the initial mass of the larva per 24 h of experiment will be our measure of foraging efficiency. The strain of the larva and the type of environment will be randomized for each trial. All larvae will be assessed at the same temperature and relative humidity.
</p>
<p align="justify">
The experimental design described above consists of a <span class="math inline">\(2\times2\)</span> factorial design. The dependent variable will be the foraging efficiency (ug/h), and the independent variables will be the larval strain (sitter-rover) and type of environment (uniform-patchy). A two-factor Analysis of Variance represents an ideal linear model to analyze the data produced by the experiments.
</p>
</div>
<div id="prediction" class="section level2" number="3.3">
<h2><span class="header-section-number">3.3</span> Prediction</h2>
<p align="justify">
We predict two possible outcomes: assuming that rover larvae have a better ability to gather more food as they forage than sitter larvae do, the former would be more efficient foragers in a patchy environment, increasing their ability to eat more food per time (Figure <a href="#fig:fig11">3.4</a>). However, if food is readily available in the environment, rovers and sitters would eat the same amount per time (Figure <a href="#fig:fig11">3.4</a>).
</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig11"></span>
<img src="index_files/figure-html/fig11-1.png" alt="Assuming there is a $GXE$ interaction, two possible outcomes are expected: In the left pannel, a rover genotype would comsume more food than a sitter genotype. In the right pannel, however, both rovers and sitters would consume the same amount of food." width="90%" />
<p class="caption">
Figure 3.4: Assuming there is a <span class="math inline">\(GXE\)</span> interaction, two possible outcomes are expected: In the left pannel, a rover genotype would comsume more food than a sitter genotype. In the right pannel, however, both rovers and sitters would consume the same amount of food.
</p>
</div>
</div>
</div>
<div id="chapter-geographic-variation-of-the-for-gene-revealed-signatures-of-local-adaptation-in-drosophila-melanogaster" class="section level1" number="4">
<h1><span class="header-section-number">4</span> Chapter: Geographic variation of the <em>for</em> gene revealed signatures of local adaptation in <em>Drosophila melanogaster</em></h1>
<div id="introduction-3" class="section level2" number="4.1">
<h2><span class="header-section-number">4.1</span> Introduction</h2>
<p align="justify">
Species distributed over broad geographic ranges face drastic variation in environmental conditions. Such species provide an opportunity to understand the mechanisms underlying evolutionary responses to environmental variation. <em>Drosophila melanogaster</em> represents a good example of a species with a widespread distribution range, being found on every continent and most islands.<span class="citation"><sup>[104]</sup></span> Since the early 1900s until the present days, this species has been a central model to understand the genetic basis of adaptation.<span class="citation"><sup>[105]</sup></span> Identifying genes underlying adaptation of populations is a key issue in evolutionary biology.<span class="citation"><sup>[106]</sup></span> For instance, the foraging gene (<em>for</em>) is best known for underlying different foraging strategies adopted by <em>D. melanogaster</em> in response to environmental variation.<span class="citation"><sup>[101]</sup></span> Detailed characterization of this gene in laboratory experiments revealed the existence of two allelic variants: rover and sitter.<span class="citation"><sup>[107]</sup></span> In the presence of nutritive yeast, larval rovers travel longer distances and eat less than sitters do. This difference is not evident in the presence of a non-nutritive agar medium,<span class="citation"><sup>[108]</sup></span> suggesting a gene-by-environment interaction.<span class="citation"><sup>[109]</sup></span> Similarly, Pereira and Sokolowski (1993)<span class="citation"><sup>[110]</sup></span> showed that adult rovers walked significantly farther from the food source after eating than did sitters, but there were no differences in walking behavior in the absence of food. This behavioral variation should also be observed in the wild, because food availability varies temporally and spatially across the range of <em>D. melanogaster</em>. Because these behavioral differences between rovers and sitters arise from genetic variation in the <em>for</em> gene, one should expect the frequency of alleles in this gene to vary over space and time.
</p>
<p align="justify">
Extensive geographic and temporal variation in various genetic markers has been identified in <em>D. melanogaster</em>.<span class="citation"><sup>[113]</sup></span> In some cases, clinal genetic variants have been directly linked to phenotypic variants.<span class="citation"><sup>[117]</sup></span> For example, associations have been discovered between the <em>Adh</em> allozyme locus and development, as well as the <em>hsp70</em> gene and heat knockdown resistance in a mid-latitude population along the east coast of Australia. There is also good evidence indicating that <em>D. melanogaster</em> evolved Bergmann‚Äôs clines on more than one continent (Figure <a href="#fig:fig12">4.1</a>). When raised in a common environment, both South American and Australian flies from high latitudes developed faster and matured at larger sizes than their counterparts from low latitudes.<span class="citation"><sup>[118]</sup></span> Similarly, seasonal stresses at high latitudes, such as overwinter survival and ephemeral food resources, impose strong selection for increased body size and somatic maintenance, whereas warm climates with ample feeding opportunities favor reduced size and decreased maintenance.<span class="citation"><sup>[119]</sup></span> Because the foraging behavior of organisms influences their ability to gather and assimilate food to fuel their maintenance requirements, one may expect the allele frequency of the <em>for</em> gene to differ in genotypes living at high latitudes and over stressful time (e.g., winter) relative to those living at lower latitudes and less stressful conditions.
</p>
<p align="justify">
Although recent studies have provided good insights on the maintenance of the <em>for</em> gene polymorphism in the laboratory, the significance and dynamics of this variation are yet to be investigated in nature. Studies based on experimental evolution suggest that density-dependent selection during the larval stage of Drosophila, rather than the adult stage, can maintain genetic variation in foraging behavior. For instance, a high larval density favors the rover variant and low larval density favors the sitter variant.<span class="citation"><sup>[120]</sup></span> Similarly, negative frequency-dependent selection promotes the increase in frequency of one variant when it is rare relative to the other variant.<span class="citation"><sup>[121]</sup></span> In addition,<span class="citation"><sup>[122]</sup></span> showed evidence of an epigenetic regulator of the <em>for</em> gene (the <em>G9a</em> methyltransferase) responsible for rover-sitter differences in adult foraging behavior. Specifically, they showed that rovers bearing <em>G9a</em> alleles, had significantly greater foraging success than sitters, but this rover‚Äìsitter difference disappeared when they carried <em>G9a</em> null alleles. In sum, researchers have now detailed advances on the mechanisms that produce variation in the Drosophila <em>for</em> gene under controlled conditions. Yet, our understanding of the environmental drivers of the genetic variation that controls the foraging behavior of flies in nature remains unknown. A good approach to fill this gap is referred to as genotype-environment association analysis,<span class="citation"><sup>[124]</sup></span> an emerging field that integrates tools from landscape genetics and population genomics to identify the environmental factors that have shaped present-day genetic variation, and the gene variants that drive local adaptation.
</p>
<p align="justify">
Here, I present results of the first genotype-environment association analysis that quantifies variation in the <em>for</em> gene among samples of <em>D. melanogaster</em> structured across space and time (Figure <a href="#fig:fig13">4.2</a>). These samples consist of published genomes of adult flies collected worldwide, and at least twice per site of collection (during spring and fall). Thus, not only could I test whether the allele frequency of the <em>for</em> gene change geographically, but also in response to seasonality. I found a distinct genetic structure among the samples collected in America and Europe. An analysis of genetic differentiation based on <span class="math inline">\(F_{st}\)</span> estimates revealed two major genetic clusters; one with all the American samples grouped together, and the other one grouping the European samples. Similarly, there is some degree of genetic differentiation among samples within each continent, suggesting a total of 6 ancestral populations (<span class="math inline">\(k\)</span>) with unique patterns of allelic variation in the <em>for</em> gene. The results indicate that different patterns of gene flow as expected under models of isolation by environment (IBE) and isolation by distance (IBD) are likely driving the genetic differentiation among populations of <em>D. melanogaster</em>. Overall, this study is essential for understanding the mechanisms underlying the evolution of foraging behavior in <em>D. melanogaster</em>.
</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig12"></span>
<img src="imgs/cline.png" alt="Latitudinal life-history clines in *D. melanogaster*. On multiple continents and subcontinents, spanning temperate to subtropical/tropical regions, fly populations exhibit major differences in fitness components across latitudes." width="50%" />
<p class="caption">
Figure 4.1: Latitudinal life-history clines in <em>D. melanogaster</em>. On multiple continents and subcontinents, spanning temperate to subtropical/tropical regions, fly populations exhibit major differences in fitness components across latitudes.
</p>
</div>
<p><br></p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig13"></span>
<img src="imgs/map-drosophila.png" alt="Available data of genome-wide allele frequency estimates from &gt;270 population-based samples of *D. melanogaster* worldwide. DGN: samples available from the Drosophila Genome Nexus project; DrosEU: samples collected by the European Drosophila Population Genomics Consortium; DrosRTEC: samples collected by the Real Time Evolution Consortium." width="100%" />
<p class="caption">
Figure 4.2: Available data of genome-wide allele frequency estimates from &gt;270 population-based samples of <em>D. melanogaster</em> worldwide. DGN: samples available from the Drosophila Genome Nexus project; DrosEU: samples collected by the European Drosophila Population Genomics Consortium; DrosRTEC: samples collected by the Real Time Evolution Consortium.
</p>
</div>
<p><br><br></p>
</div>
<div id="materials-and-methods-2" class="section level2" number="4.2">
<h2><span class="header-section-number">4.2</span> Materials and Methods</h2>
<strong>Data source</strong> <br>
<p align="justify">
The data represent a subset of a dataset assembled by the Drosophila Genome Nexus project (DGN), the European Drosophila Population Genomics Consortium (DrosEU), and the Real Time Evolution Consortium (DrosRTEC). This dataset constitutes the most comprehensive spatiotemporal sampling of <em>D. melanogaster</em> populations to date. Overall, it contains data of genome-wide allele frequency estimates from <span class="math inline">\(&gt;270\)</span> populations. Collectively, the dataset is based on sequencing of &gt;13,000 flies collected around the world and over multiple years (2003-2018). Importantly, all our analyses relied on pooled samples stored in the Genomic Data Structure (GDS) file named ‚Äúdest.all.PoolSNP.001.50.10Nov2020.ann.gds,‚Äù available at <a href="%22https://dest.bio/data-files/SNP-tables%22">https://dest.bio/data-files/SNP-tables</a>. Pooled samples from this file consist of 33‚Äì40 wild-caught males as they are more easily distinguishable morphologically from similar species than females. Detailed information on quality control, mapping, SNP calling and dataset merging can be obtained from the DEST dataset mapping pipeline <a href="%22https://dest.bio/sourcecode%22">https://dest.bio/sourcecode</a>.
</p>
<p align="justify">
I filtered the GDS object to restrict my focus on Single Nucleotide Polymorphism variants (SNPs) in the <em>for</em> gene. As annotated at <a href="%22https://flybase.org/%22">https://flybase.org/</a> (genome assembly version 6.32), the <em>for</em> gene is found on the left arm of chromosome 2 from base pairs 3,622,074-3,656,953. I further filtered the reads to get a subset of samples collected at least once during spring and winter. Because the filtering process drastically reduced the number of individuals (pools) in a few populations, I grouped together populations from the nearest neighbor localities to ensure a minimum sample size of 5. I also applied a minimum effective coverage <span class="math inline">\(N_{eff}\)</span> filter of 28 to avoid the influence of atypical observations driven by samples with low coverage.<span class="citation"><sup>[125]</sup></span> The DEST dataset provides the coordinates of each collection site, enabling one to obtain data about the seasonality of temperature and precipitation. These climatic variables were derived from the monthly values of temperature and rainfall for global land areas at a resolution of 30 s.<span class="citation"><sup>[41]</sup></span> In addition, I extracted log-transformed values of net primary production (NPP)‚Äîthe net amount of solar energy converted to plant organic matter through photosynthesis‚Äîmeasured in units of elemental carbon. This variable represents the primary source of trophic energy for the world‚Äôs ecosystems.<span class="citation"><sup>[42]</sup></span>
</p>
<strong>Quality control</strong> <br>
<p align="justify">
I removed pools and SNP loci with more than 20% missing genotypes. To do so, I used the <em>missingno</em> function from the ‚Äúpoppr‚Äù package of R v2.9.3.<span class="citation"><sup>[126]</sup></span> Subsequently, I tested for deviations from Hardy‚ÄìWeinberg equilibrium (HWE) using the function <em>hw.test</em> from the ‚Äúpegas‚Äù package of R v1.1.<span class="citation"><sup>[127]</sup></span> This function calculated p-values of an exact test with a Monte Carlo permutation of alleles (1,000 replicates). I considered loci to be out of HWE if they deviated significantly in more than 50% of populations. This decision was based on a ‚Äúfalse discovery rate‚Äù correction of the p-values.
</p>
<strong>Genetic differentiation and population structure</strong><br>
<p align="justify">
To test for genetic differentiation among populations at the level of the <em>for</em> gene, I estimated pairwise values of <span class="math inline">\(F_{st}\)</span> adopting the method suggested by Weir and Cockerham (1984)<span class="citation"><sup>[128]</sup></span>. The <span class="math inline">\(F_{st}\)</span> values serve as a tool for describing the partitioning of genetic diversity within and among populations.<span class="citation"><sup>[129]</sup></span> For example, large populations among which there is much migration tend to show little differentiation with values of <span class="math inline">\(F_{st}\)</span> close to 0, whereas small populations among which there is little migration tend to be highly differentiated with values of <span class="math inline">\(F_{st}\)</span> close to 1. I used the function <em>gl.fst.pop</em> from the package ‚ÄúdartR‚Äù v2.7.2,<span class="citation"><sup>[130]</sup></span> running 1,000 bootstraps across all loci to generate 95% confidence intervals of the values of <span class="math inline">\(F_{st}\)</span>. When the confidence intervals did not include zero, the <span class="math inline">\(F_{st}\)</span> value in question was considered significantly different from zero. To explore patterns of isolation by environment (IBE) and isolation by distance (IBD), I first performed a Partial Mantel test on genetic distance (<span class="math inline">\(F_{st}\)</span>) and primary production distance, holding geographic distance (coordinates) constant. I then performed the same test but holding the primary production distance matrix constant to account for the effect of geographic distance. To do this, I used the <em>partial.mantel</em> function from the R package ‚Äúvegan‚Äù v2.6-4, which ran a Monte-Carlo test based on 1,000 permutations and assessed the significance of the correlation between the distance matrices. I also evaluated the relationship between these distances with a Mantel correlogram, a method that does not make the assumption of linearity. For this test, I ran the function <em>mgram</em> from the R package ‚Äúecodist‚Äù v2.0.7,<span class="citation"><sup>[131]</sup></span> setting the number of permutations to 1,000 and using the Sturge‚Äôs rule to determine an appropriate number of classes. The function returns a matrix with one row for each distance class and 6 columns; two of which represent estimates of the Mantel <em>r</em>, and its associated p-value (see supporting code).
</p>
<p align="justify">
I estimated population structure using two classical approaches: an admixture analysis and a principal component analysis (PCA). I performed the admixture analysis using a sparse negative matrix factorization method (SNMF), available in the R package ‚ÄúLEA‚Äù v3.2.0.<span class="citation"><sup>[132]</sup></span> Similar to Bayesian clustering programs such as CLUSTER, the function <em>snmf</em> in LEA estimates individual admixture coefficients from the genomic matrix. Assuming <span class="math inline">\(k\)</span> ancestral populations, the function provides least-squares estimates of ancestry proportions and estimates an entropy criterion to evaluate the quality of fit of the statistical model through cross-validation. The entropy criterion enabled one to choose the number of ancestral populations that best explained the genomic data. Specifically, I used the function <em>snmf</em> in LEA to compute individual ancestry coefficients for a number of ancestral populations (<span class="math inline">\(k\)</span>), ranging between <span class="math inline">\(k=1\)</span> and <span class="math inline">\(k=10\)</span>. For each value of <span class="math inline">\(k\)</span>, I ran the algorithm 50 times and calculated the cross-entropy for each run. In addition, I performed a PCA on the genomic and environmental data using the function <em>rda</em> from the R package ‚Äúvegan‚Äù v2.6-4. This function computes the scores of a PCA and returns a scree plot for the eigenvalues of the sample covariance matrix. I identified the number of significant components for both the genomic and the environmental matrices based on a Broken Stick model. The Broken Stick model is one proposed method for estimating the number of statistically significant principal components. The idea is to model <em>N</em> variances by taking a stick of unit length and breaking it into <em>N</em> pieces by randomly selecting break points from a uniform distribution.<span class="citation"><sup>[133]</sup></span> According to the broken-stick stopping rule, principal components should be retained as long as observed eigenvalues are higher than corresponding random broken stick components. Because the number of ancestral populations is closely linked to the number of principal components that explain variation in the genomic data, this method provides a good complement to the admixture analysis, and expands one‚Äôs ability to find the most likely value of <span class="math inline">\(k\)</span>.
</p>
<strong>Detecting selection</strong> <br>
<p align="justify">
I also implemented two approaches to detect selection signatures in the data: 1) a differentiation-based method based on the OutFLANK algorithm v0.2,<span class="citation"><sup>[134]</sup></span> and 2) a genotype-environment association method based on Latent Factor Mixed Models with ridge penalties.<span class="citation"><sup>[135]</sup></span> OutFLANK calculates a likelihood on a trimmed distribution of <span class="math inline">\(F_{st}\)</span> values to infer the distribution of <span class="math inline">\(F_{st}\)</span> for neutral markers. This distribution is then used to assign q-values to each locus to detect outliers that may be due to spatially heterogeneous selection. The LFMM method is a univariate test that builds a model for each SNP and each predictor variable. Thus, it ran <span class="math inline">\(1,550\)</span> SNPs <span class="math inline">\(\times 1\)</span> synthetic predictor <span class="math inline">\(=1,550\)</span> separate LFMM tests. This method requires an estimate of the number of populations in the data (<span class="math inline">\(k\)</span>). To determine the most likely value of <span class="math inline">\(k\)</span>, I used both an admixture analysis and a PCA, as discussed earlier. Because these analyses found support for multiple <span class="math inline">\(k\)</span> values, I ran LFMMs varying <span class="math inline">\(k\)</span> accordingly. For each model fitted, I used the genomic inflation factor (GIF) to select the one that best accounted for population structure. I also applied a False Discovery Rate (FDR) of 0.05 by converting the p-values to q-values. The FDR is the expected proportion of false positives among the list of positive tests.<span class="citation"><sup>[136]</sup></span> For example, an FDR threshold of 0.05 applied to a dataset with an ideal p-value distribution would produce 5% false positives (false discoveries) among the set of positive tests. However, if the distribution of p-values deviates from the ideal distribution, this same FDR threshold of 0.05 would produce more or fewer false discoveries among the set of positive tests, depending on the skew of the distribution.
</p>
</div>
<div id="results-2" class="section level2" number="4.3">
<h2><span class="header-section-number">4.3</span> Results</h2>
<p align="justify">
In general, none of the loci nor populations were consistently out of Hardy-Weinberg equilibrium. I filtered out of the analyses only one sample that had more than 20% of missing genotypes and 62 loci that had more than 20% of missing data. The quality control procedure generated a new dataset of 1,550 loci (SNPs), and 3,100 alleles from 151 pooled samples distributed among 15 populations.
</p>
<p align="justify">
I found an overall <span class="math inline">\(F_{st}\)</span> value of 0.032, but the pairwise differences in <span class="math inline">\(F_{st}\)</span> ranged from 0 to 0.069. Interestingly, the results revealed two major clusters separating the samples collected in America from those collected in Europe (Figure <a href="#fig:fig14">4.3</a>). However, significant differentiation was also observed within each continent. The IBE and IBD analyses based on the Partial Mantel test showed a clear increase of genetic distance with NPP distance (Mantel statistic <span class="math inline">\(r=0.361\)</span>, <span class="math inline">\(p-value&lt;0.001\)</span>) and geographic distance (Mantel statistic <span class="math inline">\(r=0.926\)</span>, <span class="math inline">\(p-value&lt;0.001\)</span>). However, a smoothed local mean (read line), indicated that these relationships might not be linear (Figure <a href="#fig:fig15">4.4</a>a-b). Importantly, the Mantel correlograms supported the same pattern; populations are relatively similar at short distances, while this similarity decreases with distance (Figure <a href="#fig:fig15">4.4</a>c-d).
</p>
<p><br><br></p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig14"></span>
<img src="imgs/heatmap.jpeg" alt="Heatmap depicting genetic differentiation among populations of *D. melanogaster* based on pairwise $F_{st}$ values. In some cases, samples from adjacent localities were combined to avoid low sample sizes. Abbreviations are as follows: PA = Pennsylvania; MI-WI = Michigan and Wisconsin; NY-MA = New York and Massachusetts." width="100%" />
<p class="caption">
Figure 4.3: Heatmap depicting genetic differentiation among populations of <em>D. melanogaster</em> based on pairwise <span class="math inline">\(F_{st}\)</span> values. In some cases, samples from adjacent localities were combined to avoid low sample sizes. Abbreviations are as follows: PA = Pennsylvania; MI-WI = Michigan and Wisconsin; NY-MA = New York and Massachusetts.
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig15"></span>
<img src="imgs/isolation-by-distance.jpeg" alt="Patterns of isolation by distance and isolation by environment according to Partial Mantel tests and Correlograms. (a-b) Geographic and net primary production distances as a function of genetic distance. Colors represent estimated probability densities and the red line a smoothed local mean. (c-d) Mantel correlation at different distance classes as a function of geographic and net primary production distances. Filled dots indicate significant correlations." width="100%" />
<p class="caption">
Figure 4.4: Patterns of isolation by distance and isolation by environment according to Partial Mantel tests and Correlograms. (a-b) Geographic and net primary production distances as a function of genetic distance. Colors represent estimated probability densities and the red line a smoothed local mean. (c-d) Mantel correlation at different distance classes as a function of geographic and net primary production distances. Filled dots indicate significant correlations.
</p>
</div>
<p><br></p>
<p align="justify">
Despite the observed genetic differentiation based of the <span class="math inline">\(F_{st}\)</span> values, the OutFLANK algorithm revealed no evidence of selection. The distribution of SNPs showed no outliers that might have arisen due to spatially heterogeneous selection (Figure <a href="#fig:fig16">4.5</a>).
</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig16"></span>
<img src="imgs/OutFlank.jpeg" alt="Manhattan plot of the distribution of $F_{st}$ loci as a function of their heterozygosity. No $F_{st}$ loci showed significant deviation from the genetic background." width="100%" />
<p class="caption">
Figure 4.5: Manhattan plot of the distribution of <span class="math inline">\(F_{st}\)</span> loci as a function of their heterozygosity. No <span class="math inline">\(F_{st}\)</span> loci showed significant deviation from the genetic background.
</p>
</div>
<p><br></p>
<p align="justify">
In addition, the genetic admixture analysis suggested a pattern of genetic clusters that approximates that observed from the <span class="math inline">\(F_{st}\)</span> values. The <em>snmf</em> run with <span class="math inline">\(k=9\)</span> best accounted for population structure as it produced the lowest cross-entropy value (Figure <a href="#fig:fig17">4.6</a>a). Overall, I observed a distinct east-west gradient in population structure according to the admixture proportions (Figure <a href="#fig:fig17">4.6</a>b). This gradient seems to be stronger among the samples collected in America than those collected in Europe, although the population structure of samples from Ukraine and Russia stands out (Figure <a href="#fig:fig18">4.7</a>).
</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig17"></span>
<img src="imgs/admix-barplot.jpeg" alt="Population structure analysis based on individual ancestry coefficients for a number of ancestral populations. (a) Cross-entropy values for each *snmf* run with *k* ranging between $k=1$ and $k=10$. The red arrow indicates the most likely value of *k*. (b) Admixture proportion across populations of *D. melanogaster*. Colors indicate genetic clusters." width="100%" />
<p class="caption">
Figure 4.6: Population structure analysis based on individual ancestry coefficients for a number of ancestral populations. (a) Cross-entropy values for each <em>snmf</em> run with <em>k</em> ranging between <span class="math inline">\(k=1\)</span> and <span class="math inline">\(k=10\)</span>. The red arrow indicates the most likely value of <em>k</em>. (b) Admixture proportion across populations of <em>D. melanogaster</em>. Colors indicate genetic clusters.
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig18"></span>
<img src="imgs/map-pie.jpeg" alt="Mean admixture proportions across populations of *D. melanogaster* surveyed in America and Europe. Colors in the pie charts represent genetic clusters." width="100%" />
<p class="caption">
Figure 4.7: Mean admixture proportions across populations of <em>D. melanogaster</em> surveyed in America and Europe. Colors in the pie charts represent genetic clusters.
</p>
</div>
<p align="justify">
In contrast to the <em>snmf</em> method, a PCA of the genomic data suggested that PC1 and PC2 explained more than the random broken stick components (see supporting code). This means that <span class="math inline">\(k=2\)</span> best accounted for population structure among all samples based on a PCA assessment. Similarly, a PCA of the environmental variables showed that only the PC1 explained more than the broken stick components, with longitude, latitude, and NPP contributing to most of the variation (see supporting code). Accordingly, I selected the PC1 as a synthetic predictor variable to be used in the genotype-environment association analysis. Based on the genetic admixture analysis and the PCA, I extracted locus-specific z-scores and p-values from latent factor mixed models with <span class="math inline">\(k\)</span> ranging between 2 and 9. The GIF and adjusted p-values indicated that a good choice for the number of latent factors was <span class="math inline">\(k=6\)</span> (Figure <a href="#fig:fig19">4.8</a>a). Using <span class="math inline">\(k=6\)</span>, the default GIF correction, and an FDR threshold of 0.05, I detected 2 candidate SNPs under selection in response to the PC1 environmental predictor (Figure <a href="#fig:fig19">4.8</a>b).
</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:fig19"></span>
<img src="imgs/LFMM-results.jpeg" alt="Genotype-environment association test based on a latent factor mixed model with ridge penalty. (a) Distribution of adjusted p-values using the default genomic inflation factor. (b) Manhattan plot of loci (SNPs) potentially affected by the PC1 predictor variable. Loci highlighted in red were considered to be under selection according to a False Discovery Rate of 0.05." width="100%" />
<p class="caption">
Figure 4.8: Genotype-environment association test based on a latent factor mixed model with ridge penalty. (a) Distribution of adjusted p-values using the default genomic inflation factor. (b) Manhattan plot of loci (SNPs) potentially affected by the PC1 predictor variable. Loci highlighted in red were considered to be under selection according to a False Discovery Rate of 0.05.
</p>
</div>
</div>
<div id="discussion-2" class="section level2" number="4.4">
<h2><span class="header-section-number">4.4</span> Discussion</h2>
<p align="justify">
This study shows that populations of <em>D. melanogaster</em> are geographically differentiated based on SNP variants of the foraging gene (<em>for</em>). The significant correlation between genetic distance with both primary production distance and geographic distance indicates that genetic differentiation between populations is consistent with signatures of IBE and IBD. Accordingly, gene flow may be strong among populations that occur in close proximity and exploit similar environments. Different patterns of gene flow are expected under IBE and IDB models.<span class="citation"><sup>[137]</sup></span> One scenario predicts that with limited dispersal and no selection, genetic drift would cause populations to become more differentiated at greater distances, and that this process should be more pronounced as mean population size decreases.<span class="citation"><sup>[138]</sup></span> Thus, under strict IBD, distance predicts differentiation as a result of dispersal limitation and drift, irrespective of environmental differences. A second scenario predicts gene flow to be strong among similar environments to a greater extent than predicted under IBD. However, adaptation to local environments can disrupt patterns of IBD, and subsequently IBE could arise if maladapted immigrants from different environments are selected against, causing strong barriers to gene flow. A good example of this scenario comes from plants growing on and near mine tailings, where adaptation to different soil types has occurred.<span class="citation"><sup>[139]</sup></span> McNeilly and Antonovics (1968)<span class="citation"><sup>[140]</sup></span> showed that reproductive barriers arose through differences in flower bud development as a result of local adaptation to different soil types. Interestingly, a pattern of isolation by geographical and environmental factors has been reported recently in <em>D. melanogaster</em>.<span class="citation"><sup>[141]</sup></span> Using microsatellite genotypes, the authors showed an effect of precipitation on patterns of genetic variation across a latitudinal gradient. Similarly, a significant correlation between pairwise <span class="math inline">\(F_{st}\)</span> and geographic distance was discussed in the context of IBD in a study that examined putatively neutral SNPs, mitochondrial haplotypes, as well as inversion and transposable element insertion polymorphisms.<span class="citation"><sup>[112]</sup></span>
</p>
<p align="justify">
Although IBE and IBD may arise from local adaptation and gene flow, the outlier analysis based on genetic differentiation (OutFLANK) showed no signatures of selection. This method is based on an idea that loci influenced by directional selection will show larger genetic differentiation than neutral loci, while the opposite is true for loci subject to balancing selection.<span class="citation"><sup>[142]</sup></span> Thus, loci that exhibit unusually high or low <span class="math inline">\(F_{st}\)</span> are good candidates for being influenced by selection. OutFLANK, however, might lead to unreliable results as it is not accurate for the left tail of the distribution of <span class="math inline">\(F_{st}\)</span>‚Äîloci subject to spatially uniform balancing selection. Furthermore, OutFLANK has not been formally tested in genomic analyses involving pool-seq data, such as those analyzed in this study.<span class="citation"><sup>[134]</sup></span> The absence of selection as evident by the results of this method should then be interpreted with caution. Still, the cost-efficient pool-seq data strategy is widely applied to different fields of genomics such as the genotype-environment association analyses.<span class="citation"><sup>[144]</sup></span> While SNP calling from pools has the difficulty of distinguishing rare variants from sequencing errors, this limitation is often overcome by establishing a minimum threshold allele frequency for the analysis.<span class="citation"><sup>[145]</sup></span> In hopes of mitigating this issue, I filtered out the reads with an effective coverage (<span class="math inline">\(N_{eff}\)</span>) lower than 28 in this study. I applied this filter based on a previous work that used some of the samples included in this analysis.<span class="citation"><sup>[125]</sup></span> This study suggested that the use of samples with <span class="math inline">\(N_{eff}&lt;28\)</span> created outliers in a principal component analysis driven by <span class="math inline">\(N_{eff}\)</span>, but samples with <span class="math inline">\(N_{eff}&gt;28\)</span> no longer generated atypical observations. As a result, I avoided having a high rate of false positive variants in the analyses. On the basis of its more accurate allele frequency estimation at a lower cost than sequencing of individuals, I highlight the versatility and efficacy of pool-seq data in facilitating a broad range of genomic analyses.
</p>
<p align="justify">
In contrast to OutFLANK, the outlier analysis based on the correlation of genetic variation with environmental data showed evidence of two loci under selection. The implementation of this method involved the use of a genotype‚Äìenvironment association analysis, which identifies loci that show strong correlations with one or more environmental variables, and those loci are interpreted as potentially under selection. Accordingly, I modeled the effect of the environment on genetic variation in the <em>for</em> gene using latent factor mixed models. This approach accounts for confounding effects, when the environment is correlated with population structure. Importantly, the results suggested that NPP is one of the main drivers of genetic variation in the for gene among populations of <em>D. melanogaster</em>. NPP may be a good measure of food availability in nature as it constitutes the primary energy source for Earth‚Äôs ecosystems and complex food webs by supplying food energy to the planet‚Äôs heterotrophic organisms, such as <em>D. melanogaster</em>.<span class="citation"><sup>[146]</sup></span> If NPP varies spatially and temporally across the range of this species, genotypes better able to find food might prevail in environments with low NPP. Yet, organisms can adjust their foraging behavior to meet environmental changes in food availability.<span class="citation"><sup>[102]</sup></span> For example, northern pikes and white-tailed deers may reduce food intake during winter, when food availability is low.<span class="citation"><sup>[148]</sup></span> However, food shortages in nature may often favor increased mobility to find new food sources. Indeed, some evidence in <em>D. melanogaster</em> indicate that flies become highly active when deprived of food.<span class="citation"><sup>[149]</sup></span> In the context of the rover-sitter variants, a mechanism known as balancing selection could explain the maintenance of these variants if they are exposed to NPP heterogeneity in space and time. Although this mechanism seems plausible in Drosophila, a recent study suggested that strong balancing selection at the <em>for</em> gene beginning recently (i.e., following migration out of Africa) is unlikely.<span class="citation"><sup>[150]</sup></span>
</p>
<p align="justify">
Currently, a number of studies have demonstrated that the foraging behavior of <em>D. melanogaster</em> can be strongly influenced by the <em>for</em> gene. However, none of them have explored the environmental drivers of variation in this gene based on a genotype-environment association study. To my knowledge, this study provides the first evidence of variation in the <em>for</em> gene mediated by patterns of gene flow as expected under models of IBE and IBD. The analyses also revealed the importance of using different approaches to detect potential signatures of selection. Although all approaches used in this study suffer from either low power (differentiation-based methods) or high false-positive rates (genotype‚Äìenvironment association methods), a strategy based on the use of both types of methods seems to lead to reliable identification of outlier loci. Field studies could disentangle the contributions of selection and environmental structuring to patterns of gene flow. For example, conducting an experimental transplant study to understand how dispersal and habitat-specific selection interact to influence populations occupying heterogeneous environments. Although IBE and strong selection should promote local genetic adaptation, it is unclear how often this is the case, particularly in Drosophila.
</p>
</body>
</div>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references csl-bib-body hanging-indent" line-spacing="2">
<div id="ref-roff2002life" class="csl-entry">
1. Roff, D. A. (2002). <em>Life history evolution</em>. Sinauer Associates Inc., Sunderland.
</div>
<div id="ref-stearns1992evolution" class="csl-entry">
2. Stearns, S. C., Stearns, S. C., &amp; Press, O. U. (1992). <em>The evolution of life histories</em>. OUP Oxford. <a href="https://books.google.com/books?id=-NcNAZ06nNoC">https://books.google.com/books?id=-NcNAZ06nNoC</a>
</div>
<div id="ref-smith1974optimal" class="csl-entry">
3. Smith, C. C., &amp; Fretwell, S. D. (1974). The optimal balance between size and number of offspring. <em>The American Naturalist</em>, <em>108</em>(962), 499‚Äì506.
</div>
<div id="ref-de1992acquisition" class="csl-entry">
4. De Jong, G., &amp; Van Noordwijk, A. (1992). Acquisition and allocation of resources: Genetic (co) variances, selection, and life histories. <em>The American Naturalist</em>, <em>139</em>(4), 749‚Äì770.
</div>
<div id="ref-van1986acquisition" class="csl-entry">
5. Van Noordwijk, A. J., &amp; Jong, G. de. (1986). Acquisition and allocation of resources: Their influence on variation in life history tactics. <em>The American Naturalist</em>, <em>128</em>(1), 137‚Äì142.
</div>
<div id="ref-parker1986optimal" class="csl-entry">
6. Parker, G. A., &amp; Begon, M. (1986). Optimal egg size and clutch size: Effects of environment and maternal phenotype. <em>The American Naturalist</em>, <em>128</em>(4), 573‚Äì592.
</div>
<div id="ref-congdon1987morphological" class="csl-entry">
7. Congdon, J. D., &amp; Gibbons, J. W. (1987). Morphological constraint on egg size: A challenge to optimal egg size theory? <em>Proceedings of the National Academy of Sciences</em>, <em>84</em>(12), 4145‚Äì4147.
</div>
<div id="ref-du2005does" class="csl-entry">
8. Du, W., Ji, X., &amp; Shine, R. (2005). Does body-volume constrain reproductive output in lizards? <em>Biology Letters</em>, <em>1</em>(1), 98‚Äì100.
</div>
<div id="ref-shine1992relative" class="csl-entry">
9. Shine, R. (1992). Relative clutch mass and body shape in lizards and snakes: Is reproductive investment constrained or optimized? <em>Evolution</em>, <em>46</em>, 828‚Äì833.
</div>
<div id="ref-sinervo1991proximate" class="csl-entry">
10. Sinervo, B., &amp; Licht, P. (1991). Proximate constraints on the evolution of egg size, number, and total clutch mass in lizards. <em>Science</em>, <em>252</em>(5010), 1300‚Äì1302.
</div>
<div id="ref-vitt1978body" class="csl-entry">
11. Vitt, L. J., &amp; Congdon, J. D. (1978). Body shape, reproductive effort, and relative clutch mass in lizards: Resolution of a paradox. <em>The American Naturalist</em>, <em>112</em>(985), 595‚Äì608.
</div>
<div id="ref-oufiero2007importance" class="csl-entry">
12. Oufiero, C. E., Smith, A. J., &amp; Angilletta, M. J. (2007). The importance of energetic versus pelvic constraints on reproductive allocation by the eastern fence lizard (<em>sceloporus undulatus</em>). <em>Biological Journal of the Linnean Society</em>, <em>91</em>(3), 513‚Äì521.
</div>
<div id="ref-atkinson1994temperature" class="csl-entry">
13. Atkinson, D. (1994). Temperature and organism size: A biological law for ectotherms? <em>Adv Ecol Res</em>, <em>25</em>, 1‚Äì58.
</div>
<div id="ref-atkinson1995effects" class="csl-entry">
14. Atkinson, D. (1995). Effects of temperature on the size of aquatic ectotherms: Exceptions to the general rule. <em>Journal of Thermal Biology</em>, <em>20</em>(1-2), 61‚Äì74.
</div>
<div id="ref-atkinson1997organisms" class="csl-entry">
15. Atkinson, D., &amp; Sibly, R. M. (1997). Why are organisms usually bigger in colder environments? Making sense of a life history puzzle. <em>Trends in Ecology &amp; Evolution</em>, <em>12</em>(6), 235‚Äì239.
</div>
<div id="ref-ashton2002amphibians" class="csl-entry">
16. Ashton, K. G. (2002). Do amphibians follow bergmann‚Äôs rule? <em>Canadian Journal of Zoology</em>, <em>80</em>(4), 708‚Äì716.
</div>
<div id="ref-berrigan1994reaction" class="csl-entry">
17. Berrigan, D., &amp; Charnov, E. L. (1994). Reaction norms for age and size at maturity in response to temperature: A puzzle for life historians. <em>Oikos</em>, <em>80</em>(4), 474‚Äì478.
</div>
<div id="ref-sibly1994rearing" class="csl-entry">
18. Sibly, R., &amp; Atkinson, D. (1994). How rearing temperature affects optimal adult size in ectotherms. <em>Functional Ecology</em>, 486‚Äì493.
</div>
<div id="ref-ashton2000bergmann" class="csl-entry">
19. Ashton, K. G., Tracy, M. C., &amp; Queiroz, A. de. (2000). Is bergmann‚Äôs rule valid for mammals? <em>The American Naturalist</em>, <em>156</em>(4), 390‚Äì415.
</div>
<div id="ref-bergmann1848verhaltnisse" class="csl-entry">
20. Bergmann, C. (1848). <em><span>√ú</span>ber die verh<span>√§</span>ltnisse der w<span>√§</span>rme<span>√∂</span>konomie der thiere zu ihrer gr<span>√∂</span>sse</em>.
</div>
<div id="ref-yom2006geographic" class="csl-entry">
21. Yom-Tov, Y., &amp; Geffen, E. (2006). Geographic variation in body size: The effects of ambient temperature and precipitation. <em>Oecologia</em>, <em>148</em>(2), 213‚Äì218.
</div>
<div id="ref-mcnab2010geographic" class="csl-entry">
22. McNab, B. K. (2010). Geographic and temporal correlations of mammalian size reconsidered: A resource rule. <em>Oecologia</em>, <em>164</em>(1), 13‚Äì23.
</div>
<div id="ref-cramer1999comparing" class="csl-entry">
23. Cramer W., B. A., Kicklighter D. W., &amp; Potsdam NPP Model Intercomparison, T. P. of the. (1999). Comparing global models of terrestrial net primary productivity (NPP): Overview and key results. <em>Global Change Biology</em>, <em>5</em>(S1), 1‚Äì15.
</div>
<div id="ref-bonnet2001short" class="csl-entry">
24. Bonnet, X., Naulleau, G., Shine, R., &amp; Lourdais, O. (2001). Short-term versus long-term effects of food intake on reproductive output in a viviparous snake, <em>vipera aspis</em>. <em>Oikos</em>, <em>92</em>(2), 297‚Äì308.
</div>
<div id="ref-hahn1965fat" class="csl-entry">
25. Hahn, W. E., &amp; Tinkle, D. W. (1965). Fat body cycling and experimental evidence for its adaptive significance to ovarian follicle development in the lizard <em>uta stansburiana</em>. <em>Journal of Experimental Zoology</em>, <em>158</em>(1), 79‚Äì85.
</div>
<div id="ref-marshall2018global" class="csl-entry">
26. Marshall, D. J., Pettersen, A. K., &amp; Cameron, H. (2018). A global synthesis of offspring size variation, its eco-evolutionary causes and consequences. <em>Functional Ecology</em>, <em>32</em>(6), 1436‚Äì1446.
</div>
<div id="ref-kouyoumdjian2019transplanting" class="csl-entry">
27. Kouyoumdjian, L., Gangloff, E. J., Souchet, J., Cordero, G. A., Dupou√©, A., &amp; Aubret, F. (2019). Transplanting gravid lizards to high elevation alters maternal and embryonic oxygen physiology, but not reproductive success or hatchling phenotype. <em>Journal of Experimental Biology</em>, <em>222</em>(14), jeb206839.
</div>
<div id="ref-meiri2012ecology" class="csl-entry">
28. Meiri, S., Brown, J. H., &amp; Sibly, R. M. (2012). The ecology of lizard reproductive output. <em>Global Ecology and Biogeography</em>, <em>21</em>(5), 592‚Äì602.
</div>
<div id="ref-le2020influence" class="csl-entry">
29. Le Gouvello, D. Z., Nel, R., &amp; Cloete, A. E. (2020). The influence of individual size on clutch size and hatchling fitness traits in sea turtles. <em>Journal of Experimental Marine Biology and Ecology</em>, <em>527</em>, 151372.
</div>
<div id="ref-warner2008maternal" class="csl-entry">
30. Warner, D. A., &amp; Shine, R. (2008). Maternal nest-site choice in a lizard with temperature-dependent sex determination. <em>Animal Behaviour</em>, <em>75</em>(3), 861‚Äì870.
</div>
<div id="ref-vitt1982ecological" class="csl-entry">
31. Vitt, L. J., &amp; Price, H. J. (1982). Ecological and evolutionary determinants of relative clutch mass in lizards. <em>Herpetologica</em>, 237‚Äì255.
</div>
<div id="ref-felsenstein1985phylogenies" class="csl-entry">
32. Felsenstein, J. (1985). Phylogenies and the comparative method. <em>The American Naturalist</em>, <em>125</em>(1), 1‚Äì15.
</div>
<div id="ref-mesquita2016life" class="csl-entry">
33. Mesquita, D. O., Costa, G. C., Colli, G. R., Costa, T. B., Shepard, D. B., Vitt, L. J., &amp; Pianka, E. R. (2016). Life-history patterns of lizards of the world. <em>The American Naturalist</em>, <em>187</em>(6), 689‚Äì705.
</div>
<div id="ref-angilletta2006direct" class="csl-entry">
34. Angilletta, M. J., Oufiero, C. E., &amp; Leach√©, A. D. (2006). Direct and indirect effects of environmental temperature on the evolution of reproductive strategies: An information-theoretic approach. <em>The American Naturalist</em>, <em>168</em>(4), 123‚Äì135.
</div>
<div id="ref-meiri2008evolution" class="csl-entry">
35. Meiri, S. (2008). Evolution and ecology of lizard body sizes. <em>Global Ecology and Biogeography</em>, <em>17</em>(6), 724‚Äì734.
</div>
<div id="ref-meiri2018traits" class="csl-entry">
36. Meiri, S. (2018). Traits of lizards of the world: Variation around a successful evolutionary design. <em>Global Ecology and Biogeography</em>, <em>27</em>(10), 1168‚Äì1172.
</div>
<div id="ref-meiri2015squamate" class="csl-entry">
37. Meiri, S., Feldman, A., &amp; Kratochvƒ±ÃÅl, L. (2015). Squamate hatchling size and the evolutionary causes of negative offspring size allometry. <em>Journal of Evolutionary Biology</em>, <em>28</em>(2), 438‚Äì446.
</div>
<div id="ref-warne2008reproductive" class="csl-entry">
38. Warne, R. W., &amp; Charnov, E. L. (2008). Reproductive allometry and the size-number trade-off for lizards. <em>The American Naturalist</em>, <em>172</em>(3), 80‚Äì98.
</div>
<div id="ref-deeming2004reptilian" class="csl-entry">
39. Deeming, D. C.others. (2004). <em>Reptilian incubation: Environment, evolution and behaviour.</em> Nottingham University Press.
</div>
<div id="ref-roll2017global" class="csl-entry">
40. Roll, U., Feldman, A., Novosolov, M., Allison, A., Bauer, A. M., Bernard, R., B√∂hm, M., Castro-Herrera, F., Chirio, L., Collen, B.others. (2017). The global distribution of tetrapods reveals a need for targeted reptile conservation. <em>Nature Ecology &amp; Evolution</em>, <em>1</em>(11), 1677‚Äì1682.
</div>
<div id="ref-fick2017worldclim" class="csl-entry">
41. Fick, S. E., &amp; Hijmans, R. J. (2017). WorldClim 2: New 1-km spatial resolution climate surfaces for global land areas. <em>International Journal of Climatology</em>, <em>37</em>(12), 4302‚Äì4315.
</div>
<div id="ref-imhoff2004global" class="csl-entry">
42. Imhoff, M. L., Bounoua, L., Ricketts, T., Loucks, C., Harriss, R., &amp; Lawrence, W. T. (2004). Global patterns in human consumption of net primary production. <em>Nature</em>, <em>429</em>(6994), 870‚Äì873.
</div>
<div id="ref-van2018phylopath" class="csl-entry">
43. Bijl, W. van der. (2018). Phylopath: Easy phylogenetic path analysis in r. <em>PeerJ</em>, <em>6</em>, e4718.
</div>
<div id="ref-team2013r" class="csl-entry">
44. Team, R. C.others. (2013). <em>R: A language and environment for statistical computing</em>.
</div>
<div id="ref-zheng2016combining" class="csl-entry">
45. Zheng, Y., &amp; Wiens, J. J. (2016). Combining phylogenomic and supermatrix approaches, and a time-calibrated phylogeny for squamate reptiles (lizards and snakes) based on 52 genes and 4162 species. <em>Molecular Phylogenetics and Evolution</em>, <em>94</em>, 537‚Äì547.
</div>
<div id="ref-shipley2000new" class="csl-entry">
46. Shipley, B. (2000). A new inferential test for path models based on directed acyclic graphs. <em>Structural Equation Modeling</em>, <em>7</em>(2), 206‚Äì218.
</div>
<div id="ref-shipley2013compared" class="csl-entry">
47. Shipley, B. (2013). Compared using a d-separation test reports. <em>Ecology</em>, <em>94</em>(3), 560‚Äì564.
</div>
<div id="ref-hardenberg2013disentangling" class="csl-entry">
48. Hardenberg, A. von, &amp; Gonzalez-Voyer, A. (2013). Disentangling evolutionary cause-effect relationships with phylogenetic confirmatory path analysis. <em>Evolution: International Journal of Organic Evolution</em>, <em>67</em>(2), 378‚Äì387.
</div>
<div id="ref-burnham2002practical" class="csl-entry">
49. Burnham, K. P., &amp; Anderson, D. R. (2002). A practical information-theoretic approach. <em>Model Selection and Multimodel Inference</em>, <em>2</em>, 70‚Äì71.
</div>
<div id="ref-nevo1973adaptive" class="csl-entry">
50. Nevo, E. (1973). Adaptive variation in size of cricket frogs. <em>Ecology</em>, <em>54</em>(6), 1271‚Äì1281.
</div>
<div id="ref-olalla2009geographic" class="csl-entry">
51. Olalla-T√°rraga, M. √Å., Diniz-Filho, J. A. F., Bastos, R. P., &amp; Rodrƒ±ÃÅguez, M. √Å. (2009). Geographic body size gradients in tropical regions: Water deficit and anuran body size in the brazilian cerrado. <em>Ecography</em>, <em>32</em>(4), 581‚Äì590.
</div>
<div id="ref-gouveia2016geographical" class="csl-entry">
52. Gouveia, S. F., &amp; Correia, I. (2016). Geographical clines of body size in terrestrial amphibians: Water conservation hypothesis revisited. <em>Journal of Biogeography</em>, <em>43</em>(10), 2075‚Äì2084.
</div>
<div id="ref-pincheira2008evolution" class="csl-entry">
53. Pincheira-Donoso, D., Hodgson, D. J., &amp; Tregenza, T. (2008). The evolution of body size under environmental gradients in ectotherms: Why should bergmann‚Äôs rule apply to lizards? <em>BMC Evolutionary Biology</em>, <em>8</em>(1), 1‚Äì13.
</div>
<div id="ref-huey2003behavioral" class="csl-entry">
54. Huey, R. B., Hertz, P. E., &amp; Sinervo, B. (2003). Behavioral drive versus behavioral inertia in evolution: A null model approach. <em>The American Naturalist</em>, <em>161</em>(3), 357‚Äì366.
</div>
<div id="ref-bogert1949thermoregulation" class="csl-entry">
55. Bogert, C. M. (1949). Thermoregulation in reptiles, a factor in evolution. <em>Evolution</em>, <em>3</em>(3), 195‚Äì211.
</div>
<div id="ref-buckley2015thermoregulatory" class="csl-entry">
56. Buckley, L. B., Ehrenberger, J. C., &amp; Angilletta, M. J. (2015). Thermoregulatory behaviour limits local adaptation of thermal niches and confers sensitivity to climate change. <em>Functional Ecology</em>, <em>29</em>(8), 1038‚Äì1047.
</div>
<div id="ref-van1990evolutionary" class="csl-entry">
57. Van Damme, R., Bauwens, D., &amp; Verheyen, R. F. (1990). Evolutionary rigidity of thermal physiology: The case of the cool temperate lizard <em>lacerta vivipara</em>. <em>Oikos</em>, <em>57</em>(1), 61‚Äì67.
</div>
<div id="ref-cruz2009thermal" class="csl-entry">
58. Cruz, F. B., Belver, L., Acosta, J. C., Villavicencio, H. J., Blanco, G., &amp; C√°novas, M. G. (2009). Thermal biology of phymaturus lizards: Evolutionary constraints or lack of environmental variation? <em>Zoology</em>, <em>112</em>(6), 425‚Äì432.
</div>
<div id="ref-sunday2011global" class="csl-entry">
59. Sunday, J. M., Bates, A. E., &amp; Dulvy, N. K. (2011). Global analysis of thermal tolerance and latitude in ectotherms. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <em>278</em>(1713), 1823‚Äì1830.
</div>
<div id="ref-huey1982temperature" class="csl-entry">
60. Huey, R. B. (1982). Temperature, physiology, and the ecology of reptiles. <em>Biology of the Reptilia</em>.
</div>
<div id="ref-adolph1990influence" class="csl-entry">
61. Adolph, S. C. (1990). Influence of behavioral thermoregulation on microhabitat use by two sceloporus lizards. <em>Ecology</em>, <em>71</em>(1), 315‚Äì327.
</div>
<div id="ref-rand1964ecological" class="csl-entry">
62. Rand, A. S. (1964). Ecological distribution in anoline lizards of puerto rico. <em>Ecology</em>, <em>45</em>(4), 745‚Äì752.
</div>
<div id="ref-huey1987phylogenetic" class="csl-entry">
63. Huey, R. B., &amp; Bennett, A. F. (1987). Phylogenetic studies of coadaptation: Preferred temperatures versus optimal performance temperatures of lizards. <em>Evolution</em>, <em>41</em>(5), 1098‚Äì1115.
</div>
<div id="ref-bauwens1995evolution" class="csl-entry">
64. Bauwens, D., Garland Jr, T., Castilla, A. M., &amp; Van Damme, R. (1995). Evolution of sprint speed in lacertid lizards: Morphological, physiological, and behavioral covariation. <em>Evolution</em>, <em>49</em>(5), 848‚Äì863.
</div>
<div id="ref-huey1993evolution" class="csl-entry">
65. Huey, R. B., &amp; Kingsolver, J. G. (1993). Evolution of resistance to high temperature in ectotherms. <em>The American Naturalist</em>, <em>142</em>, 21‚Äì46.
</div>
<div id="ref-angilletta2009thermal" class="csl-entry">
66. Angilletta, M. J. (2009). <em>Thermal adaptation: A theoretical and empirical synthesis</em>. Oxford University Press, USA.
</div>
<div id="ref-partridge1996thermal" class="csl-entry">
67. Partridge, L., &amp; French, V. (1996). Thermal evolution of ectotherm body size: Why get big in the cold? <em>Seminar Series-Society for Experimetal Biology</em>, <em>59</em>, 265‚Äì292.
</div>
<div id="ref-pianka1966convexity" class="csl-entry">
68. Pianka, E. R. (1966). Convexity, desert lizards, and spatial heterogeneity. <em>Ecology</em>, <em>47</em>(6), 1055‚Äì1059.
</div>
<div id="ref-perry1999evolution" class="csl-entry">
69. Perry, G. (1999). The evolution of search modes: Ecological versus phylogenetic perspectives. <em>The American Naturalist</em>, <em>153</em>(1), 98‚Äì109.
</div>
<div id="ref-webb2003does" class="csl-entry">
70. Webb, J. K., Brook, B. W., &amp; Shine, R. (2003). Does foraging mode influence life history traits? A comparative study of growth, maturation and survival of two species of sympatric snakes from south-eastern australia. <em>Austral Ecology</em>, <em>28</em>(6), 601‚Äì610.
</div>
<div id="ref-dunham1988life" class="csl-entry">
71. Dunham, A., Miles, D., &amp; Reznick, D. (1988). Life history patterns in squamate reptiles, s. 441-522.[in:] Gans, c. <em>Huey, RB: Biology of the Reptilia</em>, <em>16</em>.
</div>
<div id="ref-huey1981ecological" class="csl-entry">
72. Huey, R. B., &amp; Pianka, E. R. (1981). Ecological consequences of foraging mode. <em>Ecology</em>, <em>62</em>(4), 991‚Äì999.
</div>
<div id="ref-anderson1988energetics" class="csl-entry">
73. Anderson, R. A., &amp; Karasov, W. H. (1988). Energetics of the lizard <em>cnemidophorus tigris</em> and life history consequences of food-acquisition mode. <em>Ecological Monographs</em>, <em>58</em>(2), 79‚Äì110.
</div>
<div id="ref-anderson1981contrasts" class="csl-entry">
74. Anderson, R., &amp; Karasov, W. (1981). Contrasts in energy intake and expenditure in sit-and-wait and widely foraging lizards. <em>Oecologia</em>, <em>49</em>(1), 67‚Äì72.
</div>
<div id="ref-nagy1984field" class="csl-entry">
75. Nagy, K. A., Huey, R. B., &amp; Bennett, A. F. (1984). Field energetics and foraging mode of kalahari lacertid lizards. <em>Ecology</em>, <em>65</em>(2), 588‚Äì596.
</div>
<div id="ref-bury2021energy" class="csl-entry">
76. Bury, S. (2021). Energy expenses on prey processing are comparable, but paid at a higher metabolic scope and for a longer time in ambush vs active predators: A multispecies study on snakes. <em>Oecologia</em>, <em>197</em>(1), 61‚Äì70.
</div>
<div id="ref-stuginski2018phylogenetic" class="csl-entry">
77. Stuginski, D. R., Navas, C. A., Barros, F. C. de, Camacho, A., Bicudo, J. E. P. W., Grego, K. F., &amp; De Carvalho, J. E. (2018). Phylogenetic analysis of standard metabolic rate of snakes: A new proposal for the understanding of interspecific variation in feeding behavior. <em>Journal of Comparative Physiology B</em>, <em>188</em>(2), 315‚Äì323.
</div>
<div id="ref-glaudas2019foraging" class="csl-entry">
78. Glaudas, X., Glennon, K. L., Martins, M., Luiselli, L., Fearn, S., Trembath, D. F., Jeliƒá, D., &amp; Alexander, G. J. (2019). Foraging mode, relative prey size and diet breadth: A phylogenetically explicit analysis of snake feeding ecology. <em>Journal of Animal Ecology</em>, <em>88</em>(5), 757‚Äì767.
</div>
<div id="ref-cooper2004tradeoffs" class="csl-entry">
79. Cooper, W. E., &amp; Per√©z-Mellado, V. (2004). Tradeoffs between escape behavior and foraging opportunity by the balearic lizard (<em>podarcis lilfordi</em>). <em>Herpetologica</em>, <em>60</em>(3), 321‚Äì324.
</div>
<div id="ref-shine1980costs" class="csl-entry">
80. Shine, R. (1980). <span>‚ÄúCosts‚Äù</span> of reproduction in reptiles. <em>Oecologia</em>, <em>46</em>(1), 92‚Äì100.
</div>
<div id="ref-silva2020reproduction" class="csl-entry">
81. Silva, K. M., Braz, H. B., Kasperoviczus, K. N., Marques, O. A., &amp; Almeida-Santos, S. M. (2020). Reproduction in the pitviper <em>bothrops jararacussu</em>: Large females increase their reproductive output while small males increase their potential to mate. <em>Zoology</em>, <em>142</em>, 125816.
</div>
<div id="ref-cooper2000tradeoffs" class="csl-entry">
82. Cooper, W. E. (2000). Tradeoffs between predation risk and feeding in a lizard, the broad-headed skink (<em>eumeces laticeps</em>). <em>Behaviour</em>, <em>137</em>(9), 1175‚Äì1189.
</div>
<div id="ref-werner1993ecological" class="csl-entry">
83. Werner, E. E., &amp; Anholt, B. R. (1993). Ecological consequences of the trade-off between growth and mortality rates mediated by foraging activity. <em>The American Naturalist</em>, <em>142</em>(2), 242‚Äì272.
</div>
<div id="ref-huey1986comparative" class="csl-entry">
84. Huey, R., &amp; Bennett, A. (1986). A comparative approach to field and laboratory studies in evolutionary biology. <em>Predator-Prey Relationshsips. University of Chicago Press, Chicago</em>, 82‚Äì98.
</div>
<div id="ref-reznick2001evolution" class="csl-entry">
85. Reznick, D., Buckwalter, G., Groff, J., &amp; Elder, D. (2001). The evolution of senescence in natural populations of guppies (<em>poecilia reticulata</em>): A comparative approach. <em>Experimental Gerontology</em>, <em>36</em>(4-6), 791‚Äì812.
</div>
<div id="ref-miles2007morphology" class="csl-entry">
86. Miles, D., Losos, J., &amp; Irschick, D. (2007). Morphology, performance, and foraging mode. <em>Lizard Ecology: The Evolutionary Consequences of Foraging Mode</em>, 49‚Äì93.
</div>
<div id="ref-peig2009new" class="csl-entry">
87. Peig, J., &amp; Green, A. J. (2009). New perspectives for estimating body condition from mass/length data: The scaled mass index as an alternative method. <em>Oikos</em>, <em>118</em>(12), 1883‚Äì1891.
</div>
<div id="ref-feldman2016body" class="csl-entry">
88. Feldman, A., Sabath, N., Pyron, R. A., Mayrose, I., &amp; Meiri, S. (2016). Body sizes and diversification rates of lizards, snakes, amphisbaenians and the tuatara. <em>Global Ecology and Biogeography</em>, <em>25</em>(2), 187‚Äì197.
</div>
<div id="ref-atchley1976statistical" class="csl-entry">
89. Atchley, W. R., Gaskins, C. T., &amp; Anderson, D. (1976). Statistical properties of ratios. I. Empirical results. <em>Systematic Zoology</em>, <em>25</em>(2), 137‚Äì148.
</div>
<div id="ref-packard1988misuse" class="csl-entry">
90. Packard, G. C., &amp; Boardman, T. J. (1988). The misuse of ratios, indices, and percentages in ecophysiological research. <em>Physiological Zoology</em>, <em>61</em>(1), 1‚Äì9.
</div>
<div id="ref-pianka2003lizards" class="csl-entry">
91. Pianka, E. R., Pianka, E. R., &amp; Vitt, L. J. (2003). <em>Lizards: Windows to the evolution of diversity</em> (Vol. 5). Univ of California Press.
</div>
<div id="ref-huelsenbeck2003stochastic" class="csl-entry">
92. Huelsenbeck, J. P., Nielsen, R., &amp; Bollback, J. P. (2003). Stochastic mapping of morphological characters. <em>Systematic Biology</em>, <em>52</em>(2), 131‚Äì158.
</div>
<div id="ref-vitt2003history" class="csl-entry">
93. Vitt, L. J., Pianka, E. R., Cooper, W. E., &amp; Schwenk, K. (2003). History and the global ecology of squamate reptiles. <em>The American Naturalist</em>, <em>162</em>(1), 44‚Äì60.
</div>
<div id="ref-fritz2010selectivity" class="csl-entry">
94. Fritz, S. A., &amp; Purvis, A. (2010). Selectivity in mammalian extinction risk and threat types: A new measure of phylogenetic signal strength in binary traits. <em>Conservation Biology</em>, <em>24</em>(4), 1042‚Äì1051.
</div>
<div id="ref-orme2013caper" class="csl-entry">
95. Orme, D., Freckleton, R., Thomas, G., Petzoldt, T., Fritz, S., Isaac, N., &amp; Pearse, W. (2013). The caper package: Comparative analysis of phylogenetics and evolution in r. <em>R Package Version</em>, <em>5</em>(2), 1‚Äì36.
</div>
<div id="ref-harvey1991comparative" class="csl-entry">
96. Harvey, P. H., Pagel, M. D.others. (1991). <em>The comparative method in evolutionary biology</em> (Vol. 239). Oxford university press Oxford.
</div>
<div id="ref-pinheiro2007linear" class="csl-entry">
97. Pinheiro, J., Bates, D., DebRoy, S., Sarkar, D., &amp; Team, R. C. (2007). Linear and nonlinear mixed effects models. <em>R Package Version</em>, <em>3</em>(57), 1‚Äì89.
</div>
<div id="ref-brown2007lizard" class="csl-entry">
98. Brown, T. K., &amp; Nagy, K. (2007). Lizard energetics and the sit-and-wait vs. Wide-foraging paradigm. <em>Lizard Ecology. Cambridge University Press, Cambridge, UK</em>, 120‚Äì140.
</div>
<div id="ref-lynch1980evolution" class="csl-entry">
99. Lynch, M. (1980). The evolution of cladoceran life histories. <em>The Quarterly Review of Biology</em>, <em>55</em>(1), 23‚Äì42.
</div>
<div id="ref-reznick1982impact" class="csl-entry">
100. Reznick, D., &amp; Endler, J. A. (1982). The impact of predation on life history evolution in trinidadian guppies (<em>poecilia reticulata</em>). <em>Evolution</em>, 160‚Äì177.
</div>
<div id="ref-anreiter2019foraging" class="csl-entry">
101. Anreiter, I., &amp; Sokolowski, M. B. (2019). The foraging gene and its behavioral effects: Pleiotropy and plasticity. <em>Annual Review of Genetics</em>, <em>53</em>, 373‚Äì392.
</div>
<div id="ref-schoener1971theory" class="csl-entry">
102. Schoener, T. W. (1971). Theory of feeding strategies. <em>Annual Review of Ecology and Systematics</em>, <em>2</em>(1), 369‚Äì404.
</div>
<div id="ref-anreiter2016foraging" class="csl-entry">
103. Anreiter, I., Vasquez, O. E., Allen, A. M., &amp; Sokolowski, M. B. (2016). Foraging path-length protocol for drosophila melanogaster larvae. <em>JoVE (Journal of Visualized Experiments)</em>, <em>110</em>, e53980.
</div>
<div id="ref-markow2005drosophila" class="csl-entry">
104. Markow, T. A., &amp; O‚ÄôGrady, P. (2005). <em>Drosophila: A guide to species identification and use</em>. Elsevier.
</div>
<div id="ref-flatt2020life" class="csl-entry">
105. Flatt, T. (2020). Life-history evolution and the genetics of fitness components in <em>drosophila melanogaster</em>. <em>Genetics</em>, <em>214</em>(1), 3‚Äì48.
</div>
<div id="ref-tenaillon2008quest" class="csl-entry">
106. Tenaillon, M. I., &amp; Tiffin, P. L. (2008). The quest for adaptive evolution: A theoretical challenge in a maze of data. <em>Current Opinion in Plant Biology</em>, <em>11</em>(2), 110‚Äì115.
</div>
<div id="ref-sokolowski1980foraging" class="csl-entry">
107. Sokolowski, M. B. (1980). Foraging strategies of <em>drosophila melanogaster</em>: A chromosomal analysis. <em>Behavior Genetics</em>, <em>10</em>, 291‚Äì302.
</div>
<div id="ref-kaun2007natural" class="csl-entry">
108. Kaun, K. R., Riedl, C. A., Chakaborty-Chatterjee, M., Belay, A. T., Douglas, S. J., Gibbs, A. G., &amp; Sokolowski, M. B. (2007). Natural variation in food acquisition mediated via a drosophila cGMP-dependent protein kinase. <em>Journal of Experimental Biology</em>, <em>210</em>(20), 3547‚Äì3558.
</div>
<div id="ref-burns2012gene" class="csl-entry">
109. Burns, J. G., Svetec, N., Rowe, L., Mery, F., Dolan, M. J., Boyce, W. T., &amp; Sokolowski, M. B. (2012). Gene‚Äìenvironment interplay in <em>drosophila melanogaster</em>: Chronic food deprivation in early life affects adult exploratory and fitness traits. <em>Proceedings of the National Academy of Sciences</em>, <em>109</em>(supplement_2), 17239‚Äì17244.
</div>
<div id="ref-pereira1993mutations" class="csl-entry">
110. Pereira, H. S., &amp; Sokolowski, M. B. (1993). Mutations in the larval foraging gene affect adult locomotory behavior after feeding in <em>drosophila melanogaster</em>. <em>Proceedings of the National Academy of Sciences</em>, <em>90</em>(11), 5044‚Äì5046.
</div>
<div id="ref-hoffmann2007climatic" class="csl-entry">
111. Hoffmann, A. A., &amp; Weeks, A. R. (2007). Climatic selection on genes and traits after a 100 year-old invasion: A critical look at the temperate-tropical clines in <em>drosophila melanogaster</em> from eastern australia. <em>Genetica</em>, <em>129</em>, 133‚Äì147.
</div>
<div id="ref-kapun2020genomic" class="csl-entry">
112. Kapun, M., Barr√≥n, M. G., Staubach, F., Obbard, D. J., Wiberg, R. A. W., Vieira, J., Goubert, C., Rota-Stabelli, O., Kankare, M., Bogaerts-M√°rquez, M.others. (2020). Genomic analysis of european <em>drosophila melanogaster</em> populations reveals longitudinal structure, continent-wide selection, and previously unknown DNA viruses. <em>Molecular Biology and Evolution</em>, <em>37</em>(9), 2661‚Äì2678.
</div>
<div id="ref-sezgin2004single" class="csl-entry">
113. Sezgin, E., Duvernell, D. D., Matzkin, L. M., Duan, Y., Zhu, C.-T., Verrelli, B. C., &amp; Eanes, W. F. (2004). Single-locus latitudinal clines and their relationship to temperate adaptation in metabolic genes and derived alleles in <em>drosophila melanogaster</em>. <em>Genetics</em>, <em>168</em>(2), 923‚Äì931.
</div>
<div id="ref-lee2013polymorphism" class="csl-entry">
114. Lee, S. F., Eyre-Walker, Y. C., Rane, R. V., Reuter, C., Vinti, G., Rako, L., Partridge, L., &amp; Hoffmann, A. A. (2013). Polymorphism in the neurofibromin gene, Nf1, is associated with antagonistic selection on wing size and development time in <em>drosophila melanogaster</em>. <em>Molecular Ecology</em>, <em>22</em>(10), 2716‚Äì2725.
</div>
<div id="ref-paaby2014highly" class="csl-entry">
115. Paaby, A. B., Bergland, A. O., Behrman, E. L., &amp; Schmidt, P. S. (2014). A highly pleiotropic amino acid polymorphism in the <em>drosophila insulin</em> receptor contributes to life-history adaptation. <em>Evolution</em>, <em>68</em>(12), 3395‚Äì3409.
</div>
<div id="ref-paaby2010identification" class="csl-entry">
116. Paaby, A. B., Blacket, M. J., Hoffmann, A. A., &amp; Schmidt, P. S. (2010). Identification of a candidate adaptive polymorphism for drosophila life history by parallel independent clines on two continents. <em>Molecular Ecology</em>, <em>19</em>(4), 760‚Äì774.
</div>
<div id="ref-schmidt2008amino" class="csl-entry">
117. Schmidt, P. S., Zhu, C.-T., Das, J., Batavia, M., Yang, L., &amp; Eanes, W. F. (2008). An amino acid polymorphism in the couch potato gene forms the basis for climatic adaptation in <em>drosophila melanogaster</em>. <em>Proceedings of the National Academy of Sciences</em>, <em>105</em>(42), 16207‚Äì16211.
</div>
<div id="ref-james1997genetic" class="csl-entry">
118. James, A. C., Azevedo, R. B., &amp; Partridge, L. (1997). Genetic and environmental responses to temperature of <em>drosophila melanogaster</em> from a latitudinal cline. <em>Genetics</em>, <em>146</em>(3), 881‚Äì890.
</div>
<div id="ref-paaby2009dissecting" class="csl-entry">
119. Paaby, A. B., &amp; Schmidt, P. S. (2009). Dissecting the genetics of longevity in <em>drosophila melanogaster</em>. <em>Fly</em>, <em>3</em>(1), 29‚Äì38.
</div>
<div id="ref-sokolowski1997evolution" class="csl-entry">
120. Sokolowski, M. B., Pereira, H. S., &amp; Hughes, K. (1997). Evolution of foraging behavior in drosophila by density-dependent selection. <em>Proceedings of the National Academy of Sciences</em>, <em>94</em>(14), 7373‚Äì7377.
</div>
<div id="ref-fitzpatrick2007maintaining" class="csl-entry">
121. Fitzpatrick, M. J., Feder, E., Rowe, L., &amp; Sokolowski, M. B. (2007). Maintaining a behaviour polymorphism by frequency-dependent selection on a single gene. <em>Nature</em>, <em>447</em>(7141), 210‚Äì212.
</div>
<div id="ref-anreiter2017epigenetic" class="csl-entry">
122. Anreiter, I., Kramer, J. M., &amp; Sokolowski, M. B. (2017). Epigenetic mechanisms modulate differences in drosophila foraging behavior. <em>Proceedings of the National Academy of Sciences</em>, <em>114</em>(47), 12518‚Äì12523.
</div>
<div id="ref-joost2007spatial" class="csl-entry">
123. Joost, S., Bonin, A., Bruford, M. W., Despr√©s, L., Conord, C., Erhardt, G., &amp; Taberlet, P. (2007). A spatial analysis method (SAM) to detect candidate loci for selection: Towards a landscape genomics approach to adaptation. <em>Molecular Ecology</em>, <em>16</em>(18), 3955‚Äì3969.
</div>
<div id="ref-rellstab2015practical" class="csl-entry">
124. Rellstab, C., Gugerli, F., Eckert, A. J., Hancock, A. M., &amp; Holderegger, R. (2015). A practical guide to environmental association analysis in landscape genomics. <em>Molecular Ecology</em>, <em>24</em>(17), 4348‚Äì4370.
</div>
<div id="ref-nunez2022cosmopolitan" class="csl-entry">
125. Nunez, J. C., Lenhart, B. A., Bangerter, A., Murray, C. S., Yu, Y., Nystrom, T. L., Tern, C., Erickson, P. A., &amp; Bergland, A. O. (2022). A cosmopolitan inversion drives seasonal adaptation in overwintering drosophila. <em>bioRxiv</em>, 2022‚Äì2012.
</div>
<div id="ref-kamvar2014poppr" class="csl-entry">
126. Kamvar, Z. N., Tabima, J. F., &amp; Gr√ºnwald, N. J. (2014). Poppr: An r package for genetic analysis of populations with clonal, partially clonal, and/or sexual reproduction. <em>PeerJ</em>, <em>2</em>, e281.
</div>
<div id="ref-paradis2010pegas" class="csl-entry">
127. Paradis, E. (2010). Pegas: An r package for population genetics with an integrated‚Äìmodular approach. <em>Bioinformatics</em>, <em>26</em>(3), 419‚Äì420.
</div>
<div id="ref-weir1984estimating" class="csl-entry">
128. Weir, B. S., &amp; Cockerham, C. C. (1984). Estimating f-statistics for the analysis of population structure. <em>Evolution</em>, 1358‚Äì1370.
</div>
<div id="ref-wright1931evolution" class="csl-entry">
129. Wright, S. (1931). Evolution in mendelian populations. <em>Genetics</em>, <em>16</em>(2), 97.
</div>
<div id="ref-gruber2018dartr" class="csl-entry">
130. Gruber, B., Unmack, P. J., Berry, O. F., &amp; Georges, A. (2018). Dartr: An r package to facilitate analysis of SNP data generated from reduced representation genome sequencing. <em>Molecular Ecology Resources</em>, <em>18</em>(3), 691‚Äì699.
</div>
<div id="ref-goslee2007ecodist" class="csl-entry">
131. Goslee, S. C., &amp; Urban, D. L. (2007). The ecodist package for dissimilarity-based analysis of ecological data. <em>Journal of Statistical Software</em>, <em>22</em>, 1‚Äì19.
</div>
<div id="ref-frichot2015lea" class="csl-entry">
132. Frichot, E., &amp; Fran√ßois, O. (2015). LEA: An r package for landscape and ecological association studies. <em>Methods in Ecology and Evolution</em>, <em>6</em>(8), 925‚Äì929.
</div>
<div id="ref-jackson1993stopping" class="csl-entry">
133. Jackson, D. A. (1993). Stopping rules in principal components analysis: A comparison of heuristical and statistical approaches. <em>Ecology</em>, <em>74</em>(8), 2204‚Äì2214.
</div>
<div id="ref-whitlock2015reliable" class="csl-entry">
134. Whitlock, M. C., &amp; Lotterhos, K. E. (2015). Reliable detection of loci responsible for local adaptation: Inference of a null model through trimming the distribution of f ST. <em>The American Naturalist</em>, <em>186</em>(S1), S24‚ÄìS36.
</div>
<div id="ref-jumentier2022sparse" class="csl-entry">
135. Jumentier, B., Caye, K., Heude, B., Lepeule, J., &amp; Fran√ßois, O. (2022). Sparse latent factor regression models for genome-wide and epigenome-wide association studies. <em>Statistical Applications in Genetics and Molecular Biology</em>, <em>21</em>(1).
</div>
<div id="ref-storey2003statistical" class="csl-entry">
136. Storey, J. D., &amp; Tibshirani, R. (2003). Statistical significance for genomewide studies. <em>Proceedings of the National Academy of Sciences</em>, <em>100</em>(16), 9440‚Äì9445.
</div>
<div id="ref-sexton2014genetic" class="csl-entry">
137. Sexton, J. P., Hangartner, S. B., &amp; Hoffmann, A. A. (2014). Genetic isolation by environment or distance: Which pattern of gene flow is most common? <em>Evolution</em>, <em>68</em>(1), 1‚Äì15.
</div>
<div id="ref-wright1943isolation" class="csl-entry">
138. Wright, S. (1943). Isolation by distance. <em>Genetics</em>, <em>28</em>(2), 114.
</div>
<div id="ref-antonovics1968evolution" class="csl-entry">
139. Antonovics, J. (1968). Evolution in closely adjacent plant populations VI. Manifold effects of gene flow. <em>Heredity</em>, <em>23</em>(4), 508‚Äì524.
</div>
<div id="ref-mcneilly1968evolution" class="csl-entry">
140. McNeilly, T., &amp; Antonovics, J. (1968). Evolution in closely adjacent plant populations IV. Barriers to gene flow. <em>Heredity</em>, <em>23</em>(2), 205‚Äì218.
</div>
<div id="ref-yue2021low" class="csl-entry">
141. Yue, L., Cao, L.-J., Chen, J.-C., Gong, Y.-J., Lin, Y.-H., Hoffmann, A. A., &amp; Wei, S.-J. (2021). Low levels of genetic differentiation with isolation by geography and environment in populations of <em>drosophila melanogaster</em> from across china. <em>Heredity</em>, <em>126</em>(6), 942‚Äì954.
</div>
<div id="ref-lewontin1973distribution" class="csl-entry">
142. Lewontin, R. C., &amp; Krakauer, J. (1973). Distribution of gene frequency as a test of the theory of the selective neutrality of polymorphisms. <em>Genetics</em>, <em>74</em>(1), 175‚Äì195.
</div>
<div id="ref-fischer2013population" class="csl-entry">
143. Fischer, M. C., Rellstab, C., Tedder, A., Zoller, S., Gugerli, F., Shimizu, K. K., Holderegger, R., &amp; Widmer, A. (2013). Population genomic footprints of selection and associations with climate in natural populations of <em>arabidopsis halleri</em> from the alps. <em>Molecular Ecology</em>, <em>22</em>(22), 5594‚Äì5607.
</div>
<div id="ref-frachon2018genomic" class="csl-entry">
144. Frachon, L., Bartoli, C., Carr√®re, S., Bouchez, O., Chaubet, A., Gautier, M., Roby, D., &amp; Roux, F. (2018). A genomic map of climate adaptation in <em>arabidopsis thaliana</em> at a micro-geographic scale. <em>Frontiers in Plant Science</em>, <em>9</em>, 967.
</div>
<div id="ref-guirao2021benchmarking" class="csl-entry">
145. Guirao-Rico, S., &amp; Gonz√°lez, J. (2021). Benchmarking the performance of pool-seq SNP callers using simulated and real sequencing data. <em>Molecular Ecology Resources</em>, <em>21</em>(4), 1216‚Äì1229.
</div>
<div id="ref-imhoff2006exploring" class="csl-entry">
146. Imhoff, M. L., &amp; Bounoua, L. (2006). Exploring global patterns of net primary production carbon supply and demand using satellite observations and statistical data. <em>Journal of Geophysical Research: Atmospheres</em>, <em>111</em>(D22).
</div>
<div id="ref-johnson1966experimental" class="csl-entry">
147. Johnson, L. (1966). Experimental determination of food consumption of pike, <em>esox lucius</em>, for growth and maintenance. <em>Journal of the Fisheries Board of Canada</em>, <em>23</em>(10), 1495‚Äì1505.
</div>
<div id="ref-silver1969fasting" class="csl-entry">
148. Silver, H., Colovos, N., Holter, J., &amp; Hayes, H. (1969). Fasting metabolism of white-tailed deer. <em>The Journal of Wildlife Management</em>, 490‚Äì498.
</div>
<div id="ref-williams2004respiratory" class="csl-entry">
149. Williams, A. E., Rose, M. R., &amp; Bradley, T. J. (2004). The respiratory pattern in <em>drosophila melanogaster</em> selected for desiccation resistance is not associated with the observed evolution of decreased locomotory activity. <em>Physiological and Biochemical Zoology</em>, <em>77</em>(1), 10‚Äì17.
</div>
<div id="ref-turner2015behavioral" class="csl-entry">
150. Turner, T., Giauque, C. C., Schrider, D. R., &amp; Kern, A. D. (2015). Behavioral variation in <em>drosophila melanogaster</em>: No evidence for common alleles of large-effect at the foraging gene in a population from north carolina, USA. <em>USA. bioRxiv</em>, <em>4325</em>.
</div>
</div>
</div>
</div>



<script>
// when the page is about to reload via servr, remember the scroll position
document.addEventListener('servr:reload', function(e) {
  sessionStorage.setItem('pagedown-scroll', window.scrollY);
});
</script>
</body>
</html>
